{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# NOTEBOOK 1 OF X OLYMPIA OYSTER DNA METHYLATION\n",
    "\n",
    "## Set Running Environment and Programs\n",
    "\n",
    "## SOFTWARE VERSION \n",
    "\n",
    " * bedtools 2.26.0\n",
    " * bismark 0.15.0  \n",
    " * blast 2.4.0+\n",
    " * bowtie2 2.2.8 \n",
    " * bsmap 2.90\n",
    " * fastqc 0.11.5  \n",
    " * R 3.2.5  \n",
    " * RStudio\n",
    " * samtools 0.1.19 \n",
    " * stacks 1.40 \n",
    " * trimmomatic 0.36\n",
    " * multiqc 0.8\n",
    " * md5sum 0.9.5\n",
    " * wget"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Set up Directory Structure"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Oly_MBDSeq\n",
    "```/Data```\n",
    "```/Notebooks```\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'/Users/hollie/Documents/Oly_Oyster_DNA_Methylation/Notebooks'"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pwd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/hollie/Documents/Oly_Oyster_DNA_Methylation\n"
     ]
    }
   ],
   "source": [
    "cd .."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: Data: File exists\r\n"
     ]
    }
   ],
   "source": [
    "!mkdir Data"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Locate and Obtain Data"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Origin of Genome file\n",
    "\n",
    "* http://owl.fish.washington.edu/halfshell/working-directory/16-10-24/Ostrea_lurida-Scaff-10k.fa\n",
    "\n",
    "\n",
    "### Origin of MBD files\n",
    "* http://owl.fish.washington.edu/nightingales/O_lurida/20160203_mbdseq/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Dowload genome file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: Data/Genome: File exists\r\n"
     ]
    }
   ],
   "source": [
    "!mkdir Data/Genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
      "100  125M  100  125M    0     0  38.7M      0  0:00:03  0:00:03 --:--:-- 44.3M\n"
     ]
    }
   ],
   "source": [
    "!curl http://owl.fish.washington.edu/halfshell/working-directory/16-10-24/Ostrea_lurida-Scaff-10k.fa > Data/Genome/Ostrea_lurida-Scaff-10k.fa"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* download checksum for genome data "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Not available"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* generate checksum for dowloaded genome data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!md5 Data/Genome/Ostrea_lurida-Scaff-10k.fa > Data/Genome/Ostrea_lurida-Scaff-10k.fa.md5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MD5 (Data/Genome/Ostrea_lurida-Scaff-10k.fa) = c7e34088dd5b97b259edbfb732872343\r\n"
     ]
    }
   ],
   "source": [
    "!cat Data/Genome/Ostrea_lurida-Scaff-10k.fa.md5"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "c7e34088dd5b97b259edbfb732872343  Ostrea_lurida-Scaff-10k.fa"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Compare genome checksum files"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Not available"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Obtain MBD Data files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: Data/MBD: File exists\r\n"
     ]
    }
   ],
   "source": [
    "!mkdir Data/MBD"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--2016-11-17 22:33:36--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data\n",
      "Resolving owl.fish.washington.edu... 128.95.149.83\n",
      "Connecting to owl.fish.washington.edu|128.95.149.83|:80... connected.\n",
      "HTTP request sent, awaiting response... 301 Moved Permanently\n",
      "Location: http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/ [following]\n",
      "--2016-11-17 22:33:40--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: unspecified [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data’\n",
      "\n",
      "owl.fish.washington     [ <=>                ]  25.48K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:33:41 (87.6 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data’ saved [26090]\n",
      "\n",
      "Loading robots.txt; please ignore errors.\n",
      "--2016-11-17 22:33:41--  http://owl.fish.washington.edu/robots.txt\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 404 Not Found\n",
      "2016-11-17 22:33:42 ERROR 404: Not Found.\n",
      "\n",
      "--2016-11-17 22:33:42--  http://owl.fish.washington.edu/icons/blank.gif\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 148 [image/gif]\n",
      "Saving to: ‘owl.fish.washington.edu/icons/blank.gif’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     148  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:33:42 (10.9 MB/s) - ‘owl.fish.washington.edu/icons/blank.gif’ saved [148/148]\n",
      "\n",
      "--2016-11-17 22:33:42--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/?C=N;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: unspecified [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=N;O=D’\n",
      "\n",
      "owl.fish.washington     [  <=>               ]  25.48K  71.7KB/s    in 0.4s    \n",
      "\n",
      "2016-11-17 22:33:42 (71.7 KB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=N;O=D’ saved [26090]\n",
      "\n",
      "--2016-11-17 22:33:42--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/?C=M;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: unspecified [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=M;O=A’\n",
      "\n",
      "owl.fish.washington     [ <=>                ]  25.48K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:33:43 (78.0 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=M;O=A’ saved [26090]\n",
      "\n",
      "--2016-11-17 22:33:43--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/?C=S;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: unspecified [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=S;O=A’\n",
      "\n",
      "owl.fish.washington     [ <=>                ]  25.48K  --.-KB/s    in 0.001s  \n",
      "\n",
      "2016-11-17 22:33:43 (30.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=S;O=A’ saved [26090]\n",
      "\n",
      "--2016-11-17 22:33:43--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/?C=D;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: unspecified [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=D;O=A’\n",
      "\n",
      "owl.fish.washington     [ <=>                ]  25.48K  --.-KB/s    in 0.001s  \n",
      "\n",
      "2016-11-17 22:33:43 (28.7 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=D;O=A’ saved [26090]\n",
      "\n",
      "--2016-11-17 22:33:43--  http://owl.fish.washington.edu/icons/back.gif\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 216 [image/gif]\n",
      "Saving to: ‘owl.fish.washington.edu/icons/back.gif’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     216  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:33:43 (9.81 MB/s) - ‘owl.fish.washington.edu/icons/back.gif’ saved [216/216]\n",
      "\n",
      "--2016-11-17 22:33:43--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 893 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     893  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:33:44 (85.2 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html’ saved [893/893]\n",
      "\n",
      "--2016-11-17 22:33:44--  http://owl.fish.washington.edu/icons/compressed.gif\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1038 (1.0K) [image/gif]\n",
      "Saving to: ‘owl.fish.washington.edu/icons/compressed.gif’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.01K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:33:44 (47.1 MB/s) - ‘owl.fish.washington.edu/icons/compressed.gif’ saved [1038/1038]\n",
      "\n",
      "--2016-11-17 22:33:44--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/checksums.md5.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 2551 (2.5K) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/checksums.md5.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   2.49K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:33:44 (221 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/checksums.md5.gz’ saved [2551/2551]\n",
      "\n",
      "--2016-11-17 22:33:44--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 248824891 (237M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 237.30M  57.0MB/s    in 7.2s    \n",
      "\n",
      "2016-11-17 22:33:51 (32.7 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s1_R1.fastq.gz’ saved [248824891/248824891]\n",
      "\n",
      "--2016-11-17 22:33:51--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 267983846 (256M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 255.57M  63.6MB/s    in 4.0s    \n",
      "\n",
      "2016-11-17 22:33:55 (64.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s2_R1.fastq.gz’ saved [267983846/267983846]\n",
      "\n",
      "--2016-11-17 22:33:55--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 222881755 (213M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 212.56M  84.5MB/s    in 2.5s    \n",
      "\n",
      "2016-11-17 22:33:58 (84.5 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s3_R1.fastq.gz’ saved [222881755/222881755]\n",
      "\n",
      "--2016-11-17 22:33:58--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 534782468 (510M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 510.01M  52.4MB/s    in 8.9s    \n",
      "\n",
      "2016-11-17 22:34:07 (57.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s4_R1.fastq.gz’ saved [534782468/534782468]\n",
      "\n",
      "--2016-11-17 22:34:07--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 530446119 (506M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 505.87M  61.0MB/s    in 8.7s    \n",
      "\n",
      "2016-11-17 22:34:17 (58.2 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s5_R1.fastq.gz’ saved [530446119/530446119]\n",
      "\n",
      "--2016-11-17 22:34:17--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 531199707 (507M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 506.59M  77.1MB/s    in 12s     \n",
      "\n",
      "2016-11-17 22:34:29 (41.4 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_1_s6_R1.fastq.gz’ saved [531199707/531199707]\n",
      "\n",
      "--2016-11-17 22:34:29--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 268244545 (256M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 255.82M  60.8MB/s    in 7.9s    \n",
      "\n",
      "2016-11-17 22:34:38 (32.5 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s1_R1.fastq.gz’ saved [268244545/268244545]\n",
      "\n",
      "--2016-11-17 22:34:38--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 243251047 (232M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 231.98M  70.9MB/s    in 3.3s    \n",
      "\n",
      "2016-11-17 22:34:42 (71.2 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s2_R1.fastq.gz’ saved [243251047/243251047]\n",
      "\n",
      "--2016-11-17 22:34:42--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 221862759 (212M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 211.58M  62.0MB/s    in 3.4s    \n",
      "\n",
      "2016-11-17 22:34:45 (62.0 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s3_R1.fastq.gz’ saved [221862759/221862759]\n",
      "\n",
      "--2016-11-17 22:34:45--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 528523381 (504M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 504.04M  62.8MB/s    in 7.2s    \n",
      "\n",
      "2016-11-17 22:34:56 (69.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s4_R1.fastq.gz’ saved [528523381/528523381]\n",
      "\n",
      "--2016-11-17 22:34:56--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 523601974 (499M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 499.35M  59.4MB/s    in 43s     \n",
      "\n",
      "2016-11-17 22:35:42 (11.6 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s5_R1.fastq.gz’ saved [523601974/523601974]\n",
      "\n",
      "--2016-11-17 22:35:42--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 525087581 (501M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 500.76M  76.3MB/s    in 6.7s    \n",
      "\n",
      "2016-11-17 22:35:50 (74.7 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_2_s6_R1.fastq.gz’ saved [525087581/525087581]\n",
      "\n",
      "--2016-11-17 22:35:50--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 276702716 (264M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 263.88M  71.2MB/s    in 3.7s    \n",
      "\n",
      "2016-11-17 22:35:54 (71.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s1_R1.fastq.gz’ saved [276702716/276702716]\n",
      "\n",
      "--2016-11-17 22:35:54--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 248360684 (237M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 236.85M  66.9MB/s    in 3.6s    \n",
      "\n",
      "2016-11-17 22:35:58 (66.7 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s2_R1.fastq.gz’ saved [248360684/248360684]\n",
      "\n",
      "--2016-11-17 22:35:58--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 224220862 (214M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 213.83M  45.7MB/s    in 6.9s    \n",
      "\n",
      "2016-11-17 22:36:05 (31.2 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s3_R1.fastq.gz’ saved [224220862/224220862]\n",
      "\n",
      "--2016-11-17 22:36:05--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 553908050 (528M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 528.25M  70.4MB/s    in 7.2s    \n",
      "\n",
      "2016-11-17 22:36:13 (73.5 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s4_R1.fastq.gz’ saved [553908050/553908050]\n",
      "\n",
      "--2016-11-17 22:36:13--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 549656752 (524M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 524.19M  14.8MB/s    in 16s     \n",
      "\n",
      "2016-11-17 22:36:30 (31.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s5_R1.fastq.gz’ saved [549656752/549656752]\n",
      "\n",
      "--2016-11-17 22:36:30--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 550446200 (525M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 524.95M  50.4MB/s    in 68s     \n",
      "\n",
      "2016-11-17 22:37:38 (7.72 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_3_s6_R1.fastq.gz’ saved [550446200/550446200]\n",
      "\n",
      "--2016-11-17 22:37:38--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 252310097 (241M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 240.62M  31.7MB/s    in 6.6s    \n",
      "\n",
      "2016-11-17 22:37:45 (36.7 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s1_R1.fastq.gz’ saved [252310097/252310097]\n",
      "\n",
      "--2016-11-17 22:37:45--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 231316479 (221M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 220.60M  7.28MB/s    in 21s     \n",
      "\n",
      "2016-11-17 22:38:07 (10.5 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s2_R1.fastq.gz’ saved [231316479/231316479]\n",
      "\n",
      "--2016-11-17 22:38:07--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 212333266 (202M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 202.50M  62.0MB/s    in 3.3s    \n",
      "\n",
      "2016-11-17 22:38:11 (61.5 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s3_R1.fastq.gz’ saved [212333266/212333266]\n",
      "\n",
      "--2016-11-17 22:38:11--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 515865805 (492M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 491.97M  53.7MB/s    in 9.6s    \n",
      "\n",
      "2016-11-17 22:38:21 (51.0 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s4_R1.fastq.gz’ saved [515865805/515865805]\n",
      "\n",
      "--2016-11-17 22:38:21--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 510537400 (487M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 486.89M  50.0MB/s    in 11s     \n",
      "\n",
      "2016-11-17 22:38:32 (45.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s5_R1.fastq.gz’ saved [510537400/510537400]\n",
      "\n",
      "--2016-11-17 22:38:32--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 510848218 (487M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 487.18M  40.1MB/s    in 8.7s    \n",
      "\n",
      "2016-11-17 22:38:42 (56.1 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_4_s6_R1.fastq.gz’ saved [510848218/510848218]\n",
      "\n",
      "--2016-11-17 22:38:42--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 270191373 (258M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 257.67M  57.2MB/s    in 4.6s    \n",
      "\n",
      "2016-11-17 22:38:47 (56.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s1_R1.fastq.gz’ saved [270191373/270191373]\n",
      "\n",
      "--2016-11-17 22:38:47--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 251796994 (240M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 240.13M  32.5MB/s    in 6.0s    \n",
      "\n",
      "2016-11-17 22:38:53 (39.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s2_R1.fastq.gz’ saved [251796994/251796994]\n",
      "\n",
      "--2016-11-17 22:38:53--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 226130762 (216M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 215.65M  62.3MB/s    in 5.0s    \n",
      "\n",
      "2016-11-17 22:38:59 (43.5 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s3_R1.fastq.gz’ saved [226130762/226130762]\n",
      "\n",
      "--2016-11-17 22:38:59--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 542841990 (518M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 517.69M  36.2MB/s    in 13s     \n",
      "\n",
      "2016-11-17 22:39:13 (40.4 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s4_R1.fastq.gz’ saved [542841990/542841990]\n",
      "\n",
      "--2016-11-17 22:39:13--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 537326780 (512M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 512.43M  69.0MB/s    in 8.4s    \n",
      "\n",
      "2016-11-17 22:39:21 (60.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s5_R1.fastq.gz’ saved [537326780/537326780]\n",
      "\n",
      "--2016-11-17 22:39:21--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 537889685 (513M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 512.97M  56.2MB/s    in 11s     \n",
      "\n",
      "2016-11-17 22:39:33 (48.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_5_s6_R1.fastq.gz’ saved [537889685/537889685]\n",
      "\n",
      "--2016-11-17 22:39:33--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 249986608 (238M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 238.41M  68.3MB/s    in 3.5s    \n",
      "\n",
      "2016-11-17 22:39:37 (67.4 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s1_R1.fastq.gz’ saved [249986608/249986608]\n",
      "\n",
      "--2016-11-17 22:39:37--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 233478143 (223M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 222.66M  36.2MB/s    in 5.5s    \n",
      "\n",
      "2016-11-17 22:39:43 (40.1 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s2_R1.fastq.gz’ saved [233478143/233478143]\n",
      "\n",
      "--2016-11-17 22:39:43--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 201033784 (192M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 191.72M  52.4MB/s    in 3.7s    \n",
      "\n",
      "2016-11-17 22:39:47 (52.4 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s3_R1.fastq.gz’ saved [201033784/201033784]\n",
      "\n",
      "--2016-11-17 22:39:47--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 530632020 (506M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 506.05M  35.6MB/s    in 14s     \n",
      "\n",
      "2016-11-17 22:40:01 (36.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s4_R1.fastq.gz’ saved [530632020/530632020]\n",
      "\n",
      "--2016-11-17 22:40:01--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 525962590 (502M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 501.60M  7.69MB/s    in 25s     \n",
      "\n",
      "2016-11-17 22:40:31 (20.0 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s5_R1.fastq.gz’ saved [525962590/525962590]\n",
      "\n",
      "--2016-11-17 22:40:31--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 526447787 (502M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 502.06M  48.1MB/s    in 11s     \n",
      "\n",
      "2016-11-17 22:40:56 (44.0 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_6_s6_R1.fastq.gz’ saved [526447787/526447787]\n",
      "\n",
      "--2016-11-17 22:40:56--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 286933246 (274M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 273.64M  69.9MB/s    in 3.9s    \n",
      "\n",
      "2016-11-17 22:41:01 (70.7 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s1_R1.fastq.gz’ saved [286933246/286933246]\n",
      "\n",
      "--2016-11-17 22:41:01--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 262633261 (250M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 250.47M  69.6MB/s    in 3.7s    \n",
      "\n",
      "2016-11-17 22:41:05 (67.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s2_R1.fastq.gz’ saved [262633261/262633261]\n",
      "\n",
      "--2016-11-17 22:41:05--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 234659110 (224M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 223.79M  70.3MB/s    in 3.2s    \n",
      "\n",
      "2016-11-17 22:41:08 (70.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s3_R1.fastq.gz’ saved [234659110/234659110]\n",
      "\n",
      "--2016-11-17 22:41:08--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 571797111 (545M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 545.31M  56.8MB/s    in 11s     \n",
      "\n",
      "2016-11-17 22:41:20 (49.6 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s4_R1.fastq.gz’ saved [571797111/571797111]\n",
      "\n",
      "--2016-11-17 22:41:20--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 565521288 (539M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 539.32M  54.7MB/s    in 9.9s    \n",
      "\n",
      "2016-11-17 22:41:30 (54.7 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s5_R1.fastq.gz’ saved [565521288/565521288]\n",
      "\n",
      "--2016-11-17 22:41:30--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 565987919 (540M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 539.77M  30.9MB/s    in 15s     \n",
      "\n",
      "2016-11-17 22:41:45 (36.4 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_7_s6_R1.fastq.gz’ saved [565987919/565987919]\n",
      "\n",
      "--2016-11-17 22:41:45--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 233845298 (223M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 223.01M  71.3MB/s    in 3.1s    \n",
      "\n",
      "2016-11-17 22:41:49 (71.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s1_R1.fastq.gz’ saved [233845298/233845298]\n",
      "\n",
      "--2016-11-17 22:41:49--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 213559090 (204M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 203.67M  73.9MB/s    in 2.8s    \n",
      "\n",
      "2016-11-17 22:41:52 (73.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s2_R1.fastq.gz’ saved [213559090/213559090]\n",
      "\n",
      "--2016-11-17 22:41:52--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 192603771 (184M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 183.68M  69.9MB/s    in 2.6s    \n",
      "\n",
      "2016-11-17 22:41:55 (69.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s3_R1.fastq.gz’ saved [192603771/192603771]\n",
      "\n",
      "--2016-11-17 22:41:55--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 444567938 (424M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 423.97M  40.3MB/s    in 8.5s    \n",
      "\n",
      "2016-11-17 22:42:03 (49.7 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s4_R1.fastq.gz’ saved [444567938/444567938]\n",
      "\n",
      "--2016-11-17 22:42:03--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 440349502 (420M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 419.95M  61.7MB/s    in 6.2s    \n",
      "\n",
      "2016-11-17 22:42:10 (67.4 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s5_R1.fastq.gz’ saved [440349502/440349502]\n",
      "\n",
      "--2016-11-17 22:42:10--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 441283499 (421M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 420.84M  53.5MB/s    in 7.3s    \n",
      "\n",
      "2016-11-17 22:42:18 (57.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_8_s6_R1.fastq.gz’ saved [441283499/441283499]\n",
      "\n",
      "--2016-11-17 22:42:18--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 424497717 (405M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 404.83M  14.0MB/s    in 25s     \n",
      "\n",
      "2016-11-17 22:42:43 (16.2 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s1_R1.fastq.gz’ saved [424497717/424497717]\n",
      "\n",
      "--2016-11-17 22:42:43--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 405389352 (387M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 386.61M  12.2MB/s    in 14s     \n",
      "\n",
      "2016-11-17 22:42:57 (28.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s2_R1.fastq.gz’ saved [405389352/405389352]\n",
      "\n",
      "--2016-11-17 22:42:57--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 356396053 (340M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 339.89M  38.9MB/s    in 7.2s    \n",
      "\n",
      "2016-11-17 22:43:05 (47.4 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s3_R1.fastq.gz’ saved [356396053/356396053]\n",
      "\n",
      "--2016-11-17 22:43:05--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 877242241 (837M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 836.60M  75.7MB/s    in 11s     \n",
      "\n",
      "2016-11-17 22:43:16 (78.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s4_R1.fastq.gz’ saved [877242241/877242241]\n",
      "\n",
      "--2016-11-17 22:43:16--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 869744042 (829M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 829.45M  59.4MB/s    in 18s     \n",
      "\n",
      "2016-11-17 22:43:34 (46.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s5_R1.fastq.gz’ saved [869744042/869744042]\n",
      "\n",
      "--2016-11-17 22:43:34--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 871128656 (831M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 830.77M  51.9MB/s    in 12s     \n",
      "\n",
      "2016-11-17 22:43:47 (67.6 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_9_s6_R1.fastq.gz’ saved [871128656/871128656]\n",
      "\n",
      "--2016-11-17 22:43:47--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 461786453 (440M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 440.39M  47.9MB/s    in 11s     \n",
      "\n",
      "2016-11-17 22:43:59 (38.5 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s1_R1.fastq.gz’ saved [461786453/461786453]\n",
      "\n",
      "--2016-11-17 22:43:59--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 425525692 (406M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 405.81M  75.0MB/s    in 6.7s    \n",
      "\n",
      "2016-11-17 22:44:06 (60.5 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s2_R1.fastq.gz’ saved [425525692/425525692]\n",
      "\n",
      "--2016-11-17 22:44:06--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 386595300 (369M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 368.69M  72.9MB/s    in 4.9s    \n",
      "\n",
      "2016-11-17 22:44:11 (74.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s3_R1.fastq.gz’ saved [386595300/386595300]\n",
      "\n",
      "--2016-11-17 22:44:11--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 891629367 (850M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 850.32M  73.4MB/s    in 14s     \n",
      "\n",
      "2016-11-17 22:44:25 (61.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s4_R1.fastq.gz’ saved [891629367/891629367]\n",
      "\n",
      "--2016-11-17 22:44:25--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 900453435 (859M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 858.74M  51.4MB/s    in 13s     \n",
      "\n",
      "2016-11-17 22:44:40 (64.4 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s5_R1.fastq.gz’ saved [900453435/900453435]\n",
      "\n",
      "--2016-11-17 22:44:40--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 894105873 (853M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 852.69M  7.81MB/s    in 34s     \n",
      "\n",
      "2016-11-17 22:45:15 (25.2 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_10_s6_R1.fastq.gz’ saved [894105873/894105873]\n",
      "\n",
      "--2016-11-17 22:45:15--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 301891742 (288M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 287.91M  71.5MB/s    in 4.1s    \n",
      "\n",
      "2016-11-17 22:45:20 (70.6 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s1_R1.fastq.gz’ saved [301891742/301891742]\n",
      "\n",
      "--2016-11-17 22:45:20--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 278809599 (266M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 265.89M  65.0MB/s    in 4.0s    \n",
      "\n",
      "2016-11-17 22:45:24 (66.2 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s2_R1.fastq.gz’ saved [278809599/278809599]\n",
      "\n",
      "--2016-11-17 22:45:24--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 249509388 (238M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 237.95M  29.7MB/s    in 12s     \n",
      "\n",
      "2016-11-17 22:45:37 (19.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s3_R1.fastq.gz’ saved [249509388/249509388]\n",
      "\n",
      "--2016-11-17 22:45:37--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 629943224 (601M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 600.76M  77.0MB/s    in 8.4s    \n",
      "\n",
      "2016-11-17 22:45:47 (71.1 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s4_R1.fastq.gz’ saved [629943224/629943224]\n",
      "\n",
      "--2016-11-17 22:45:47--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 623967219 (595M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 595.06M  70.7MB/s    in 8.2s    \n",
      "\n",
      "2016-11-17 22:45:56 (72.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s5_R1.fastq.gz’ saved [623967219/623967219]\n",
      "\n",
      "--2016-11-17 22:45:56--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 624617819 (596M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 595.68M  71.2MB/s    in 11s     \n",
      "\n",
      "2016-11-17 22:46:07 (55.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_11_s6_R1.fastq.gz’ saved [624617819/624617819]\n",
      "\n",
      "--2016-11-17 22:46:07--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 366344360 (349M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 349.37M  61.8MB/s    in 9.5s    \n",
      "\n",
      "2016-11-17 22:46:17 (36.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s1_R1.fastq.gz’ saved [366344360/366344360]\n",
      "\n",
      "--2016-11-17 22:46:17--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 337253424 (322M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 321.63M  57.9MB/s    in 5.2s    \n",
      "\n",
      "2016-11-17 22:46:22 (61.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s2_R1.fastq.gz’ saved [337253424/337253424]\n",
      "\n",
      "--2016-11-17 22:46:22--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 304187807 (290M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 290.10M  34.1MB/s    in 7.1s    \n",
      "\n",
      "2016-11-17 22:46:30 (40.7 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s3_R1.fastq.gz’ saved [304187807/304187807]\n",
      "\n",
      "--2016-11-17 22:46:30--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 680121587 (649M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 648.61M  69.8MB/s    in 9.6s    \n",
      "\n",
      "2016-11-17 22:46:40 (67.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s4_R1.fastq.gz’ saved [680121587/680121587]\n",
      "\n",
      "--2016-11-17 22:46:40--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 674084111 (643M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 642.86M  76.6MB/s    in 9.5s    \n",
      "\n",
      "2016-11-17 22:46:50 (68.0 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s5_R1.fastq.gz’ saved [674084111/674084111]\n",
      "\n",
      "--2016-11-17 22:46:50--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 672601082 (641M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 641.44M  34.3MB/s    in 15s     \n",
      "\n",
      "2016-11-17 22:47:05 (42.7 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_12_s6_R1.fastq.gz’ saved [672601082/672601082]\n",
      "\n",
      "--2016-11-17 22:47:05--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 373081728 (356M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 355.80M  48.7MB/s    in 7.0s    \n",
      "\n",
      "2016-11-17 22:47:14 (51.0 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s1_R1.fastq.gz’ saved [373081728/373081728]\n",
      "\n",
      "--2016-11-17 22:47:14--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 344931090 (329M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 328.95M  59.6MB/s    in 5.5s    \n",
      "\n",
      "2016-11-17 22:47:20 (59.5 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s2_R1.fastq.gz’ saved [344931090/344931090]\n",
      "\n",
      "--2016-11-17 22:47:20--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 312968086 (298M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 298.47M  73.2MB/s    in 4.0s    \n",
      "\n",
      "2016-11-17 22:47:24 (75.1 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s3_R1.fastq.gz’ saved [312968086/312968086]\n",
      "\n",
      "--2016-11-17 22:47:24--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 729047866 (695M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 695.27M  56.3MB/s    in 34s     \n",
      "\n",
      "2016-11-17 22:47:59 (20.5 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s4_R1.fastq.gz’ saved [729047866/729047866]\n",
      "\n",
      "--2016-11-17 22:47:59--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 736721942 (703M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 702.59M  44.4MB/s    in 14s     \n",
      "\n",
      "2016-11-17 22:48:13 (50.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s5_R1.fastq.gz’ saved [736721942/736721942]\n",
      "\n",
      "--2016-11-17 22:48:13--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 731037702 (697M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 697.17M  58.9MB/s    in 37s     \n",
      "\n",
      "2016-11-17 22:48:59 (18.7 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_13_s6_R1.fastq.gz’ saved [731037702/731037702]\n",
      "\n",
      "--2016-11-17 22:48:59--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 508635453 (485M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 485.07M  82.5MB/s    in 5.8s    \n",
      "\n",
      "2016-11-17 22:49:06 (83.1 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s1_R1.fastq.gz’ saved [508635453/508635453]\n",
      "\n",
      "--2016-11-17 22:49:06--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 472813317 (451M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 450.91M  64.8MB/s    in 6.6s    \n",
      "\n",
      "2016-11-17 22:49:13 (68.4 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s2_R1.fastq.gz’ saved [472813317/472813317]\n",
      "\n",
      "--2016-11-17 22:49:13--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 435948207 (416M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 415.75M  36.5MB/s    in 7.5s    \n",
      "\n",
      "2016-11-17 22:49:21 (55.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s3_R1.fastq.gz’ saved [435948207/435948207]\n",
      "\n",
      "--2016-11-17 22:49:21--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1003679229 (957M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 957.18M  9.19MB/s    in 25s     \n",
      "\n",
      "2016-11-17 22:49:45 (38.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s4_R1.fastq.gz’ saved [1003679229/1003679229]\n",
      "\n",
      "--2016-11-17 22:49:45--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 994579450 (949M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 948.50M  61.6MB/s    in 20s     \n",
      "\n",
      "2016-11-17 22:50:06 (48.2 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s5_R1.fastq.gz’ saved [994579450/994579450]\n",
      "\n",
      "--2016-11-17 22:50:06--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 993164001 (947M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 947.15M  61.8MB/s    in 14s     \n",
      "\n",
      "2016-11-17 22:50:21 (65.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_14_s6_R1.fastq.gz’ saved [993164001/993164001]\n",
      "\n",
      "--2016-11-17 22:50:21--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 413649667 (394M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 394.49M  46.9MB/s    in 6.6s    \n",
      "\n",
      "2016-11-17 22:50:28 (59.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s1_R1.fastq.gz’ saved [413649667/413649667]\n",
      "\n",
      "--2016-11-17 22:50:28--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 380012651 (362M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 362.41M  53.6MB/s    in 6.7s    \n",
      "\n",
      "2016-11-17 22:50:35 (53.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s2_R1.fastq.gz’ saved [380012651/380012651]\n",
      "\n",
      "--2016-11-17 22:50:35--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 345264361 (329M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 329.27M  17.1MB/s    in 11s     \n",
      "\n",
      "2016-11-17 22:50:46 (31.2 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s3_R1.fastq.gz’ saved [345264361/345264361]\n",
      "\n",
      "--2016-11-17 22:50:46--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 781005523 (745M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 744.82M  58.3MB/s    in 34s     \n",
      "\n",
      "2016-11-17 22:51:21 (21.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s4_R1.fastq.gz’ saved [781005523/781005523]\n",
      "\n",
      "--2016-11-17 22:51:21--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s5_R1.fastq.gz\n",
      "Connecting to owl.fish.washington.edu|128.95.149.83|:80... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 773232610 (737M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 737.41M  30.2MB/s    in 15s     \n",
      "\n",
      "2016-11-17 22:51:36 (50.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s5_R1.fastq.gz’ saved [773232610/773232610]\n",
      "\n",
      "--2016-11-17 22:51:36--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 775343721 (739M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 739.42M  75.6MB/s    in 29s     \n",
      "\n",
      "2016-11-17 22:52:05 (25.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_15_s6_R1.fastq.gz’ saved [775343721/775343721]\n",
      "\n",
      "--2016-11-17 22:52:05--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 427753017 (408M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 407.94M  54.6MB/s    in 16s     \n",
      "\n",
      "2016-11-17 22:52:22 (26.1 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s1_R1.fastq.gz’ saved [427753017/427753017]\n",
      "\n",
      "--2016-11-17 22:52:22--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 393965052 (376M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 375.71M  15.0MB/s    in 32s     \n",
      "\n",
      "2016-11-17 22:52:54 (11.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s2_R1.fastq.gz’ saved [393965052/393965052]\n",
      "\n",
      "--2016-11-17 22:52:54--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 356132654 (340M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 339.63M  75.7MB/s    in 4.8s    \n",
      "\n",
      "2016-11-17 22:53:00 (71.4 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s3_R1.fastq.gz’ saved [356132654/356132654]\n",
      "\n",
      "--2016-11-17 22:53:00--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 850651708 (811M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 811.24M  76.3MB/s    in 13s     \n",
      "\n",
      "2016-11-17 22:53:13 (63.5 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s4_R1.fastq.gz’ saved [850651708/850651708]\n",
      "\n",
      "--2016-11-17 22:53:13--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 859684798 (820M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 819.86M  85.6MB/s    in 10s     \n",
      "\n",
      "2016-11-17 22:53:23 (79.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s5_R1.fastq.gz’ saved [859684798/859684798]\n",
      "\n",
      "--2016-11-17 22:53:23--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 851582328 (812M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 812.13M  41.6MB/s    in 20s     \n",
      "\n",
      "2016-11-17 22:53:44 (40.4 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_16_s6_R1.fastq.gz’ saved [851582328/851582328]\n",
      "\n",
      "--2016-11-17 22:53:44--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 241084811 (230M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 229.92M  61.1MB/s    in 3.7s    \n",
      "\n",
      "2016-11-17 22:53:48 (61.6 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s1_R1.fastq.gz’ saved [241084811/241084811]\n",
      "\n",
      "--2016-11-17 22:53:48--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 218947818 (209M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 208.80M  74.5MB/s    in 2.8s    \n",
      "\n",
      "2016-11-17 22:53:56 (74.5 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s2_R1.fastq.gz’ saved [218947818/218947818]\n",
      "\n",
      "--2016-11-17 22:53:56--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 195538921 (186M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 186.48M  38.8MB/s    in 5.0s    \n",
      "\n",
      "2016-11-17 22:54:01 (37.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s3_R1.fastq.gz’ saved [195538921/195538921]\n",
      "\n",
      "--2016-11-17 22:54:01--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 490084117 (467M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 467.38M  70.2MB/s    in 6.7s    \n",
      "\n",
      "2016-11-17 22:54:08 (69.3 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s4_R1.fastq.gz’ saved [490084117/490084117]\n",
      "\n",
      "--2016-11-17 22:54:08--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 485178293 (463M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 462.70M  50.9MB/s    in 8.7s    \n",
      "\n",
      "2016-11-17 22:54:17 (53.2 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s5_R1.fastq.gz’ saved [485178293/485178293]\n",
      "\n",
      "--2016-11-17 22:54:17--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 486023694 (464M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 463.51M  42.0MB/s    in 22s     \n",
      "\n",
      "2016-11-17 22:54:40 (20.8 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_17_s6_R1.fastq.gz’ saved [486023694/486023694]\n",
      "\n",
      "--2016-11-17 22:54:40--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s1_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 310631053 (296M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s1_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 296.24M  66.3MB/s    in 4.3s    \n",
      "\n",
      "2016-11-17 22:54:45 (69.1 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s1_R1.fastq.gz’ saved [310631053/310631053]\n",
      "\n",
      "--2016-11-17 22:54:45--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s2_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 290011334 (277M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s2_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 276.58M  67.3MB/s    in 5.0s    \n",
      "\n",
      "2016-11-17 22:54:50 (55.4 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s2_R1.fastq.gz’ saved [290011334/290011334]\n",
      "\n",
      "--2016-11-17 22:54:50--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s3_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 251403194 (240M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s3_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 239.76M  72.0MB/s    in 3.4s    \n",
      "\n",
      "2016-11-17 22:54:54 (71.4 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s3_R1.fastq.gz’ saved [251403194/251403194]\n",
      "\n",
      "--2016-11-17 22:54:54--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s4_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 627466819 (598M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s4_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 598.40M  6.05MB/s    in 47s     \n",
      "\n",
      "2016-11-17 22:55:42 (12.7 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s4_R1.fastq.gz’ saved [627466819/627466819]\n",
      "\n",
      "--2016-11-17 22:55:42--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s5_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 621309934 (593M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s5_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 592.53M  56.1MB/s    in 11s     \n",
      "\n",
      "2016-11-17 22:55:53 (56.4 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s5_R1.fastq.gz’ saved [621309934/621309934]\n",
      "\n",
      "--2016-11-17 22:55:53--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s6_R1.fastq.gz\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 622446787 (594M) [application/x-gzip]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s6_R1.fastq.gz’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 593.61M  71.2MB/s    in 9.0s    \n",
      "\n",
      "2016-11-17 22:56:03 (66.0 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/zr1394_18_s6_R1.fastq.gz’ saved [622446787/622446787]\n",
      "\n",
      "--2016-11-17 22:56:03--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/?C=N;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: unspecified [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=N;O=A’\n",
      "\n",
      "owl.fish.washington     [ <=>                ]  25.48K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (106 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=N;O=A’ saved [26090]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/?C=M;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: unspecified [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=M;O=D’\n",
      "\n",
      "owl.fish.washington     [ <=>                ]  25.48K  --.-KB/s    in 0.001s  \n",
      "\n",
      "2016-11-17 22:56:04 (23.7 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=M;O=D’ saved [26090]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/?C=S;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: unspecified [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=S;O=D’\n",
      "\n",
      "owl.fish.washington     [ <=>                ]  25.48K  --.-KB/s    in 0.001s  \n",
      "\n",
      "2016-11-17 22:56:04 (30.6 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=S;O=D’ saved [26090]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/?C=D;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: unspecified [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=D;O=D’\n",
      "\n",
      "owl.fish.washington     [ <=>                ]  25.48K  --.-KB/s    in 0.001s  \n",
      "\n",
      "2016-11-17 22:56:04 (29.6 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/index.html?C=D;O=D’ saved [26090]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/?C=N;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 893 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=N;O=D’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     893  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (106 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=N;O=D’ saved [893/893]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/?C=M;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 893 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=M;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     893  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (106 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=M;O=A’ saved [893/893]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/?C=S;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 893 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=S;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     893  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (65.5 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=S;O=A’ saved [893/893]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/?C=D;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 893 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=D;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     893  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (85.2 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=D;O=A’ saved [893/893]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1526 (1.5K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/index.html’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.49K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (162 MB/s) - ‘owl.fish.washington.edu/symbiodinium/index.html’ saved [1526/1526]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/icons/folder.gif\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 225 [image/gif]\n",
      "Saving to: ‘owl.fish.washington.edu/icons/folder.gif’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     225  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (23.8 MB/s) - ‘owl.fish.washington.edu/icons/folder.gif’ saved [225/225]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/?C=N;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 893 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=N;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     893  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (106 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=N;O=A’ saved [893/893]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/?C=M;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 893 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=M;O=D’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     893  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (94.6 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=M;O=D’ saved [893/893]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/?C=S;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 893 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=S;O=D’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     893  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (94.6 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=S;O=D’ saved [893/893]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/Oly_MBD/?C=D;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 893 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=D;O=D’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     893  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (94.6 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBD/index.html?C=D;O=D’ saved [893/893]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/?C=N;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1526 (1.5K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/index.html?C=N;O=D’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.49K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (182 MB/s) - ‘owl.fish.washington.edu/symbiodinium/index.html?C=N;O=D’ saved [1526/1526]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/?C=M;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1526 (1.5K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/index.html?C=M;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.49K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (162 MB/s) - ‘owl.fish.washington.edu/symbiodinium/index.html?C=M;O=A’ saved [1526/1526]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/?C=S;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1526 (1.5K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/index.html?C=S;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.49K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (146 MB/s) - ‘owl.fish.washington.edu/symbiodinium/index.html?C=S;O=A’ saved [1526/1526]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/?C=D;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1526 (1.5K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/index.html?C=D;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.49K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (146 MB/s) - ‘owl.fish.washington.edu/symbiodinium/index.html?C=D;O=A’ saved [1526/1526]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 9195 (9.0K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/index.html’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   8.98K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (106 MB/s) - ‘owl.fish.washington.edu/index.html’ saved [9195/9195]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 924 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     924  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (97.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html’ saved [924/924]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/icons/unknown.gif\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 245 [image/gif]\n",
      "Saving to: ‘owl.fish.washington.edu/icons/unknown.gif’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     245  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (29.2 MB/s) - ‘owl.fish.washington.edu/icons/unknown.gif’ saved [245/245]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/Oly_MBDseq_Analysis.ipynb\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 422787 (413K) [text/plain]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Oly_MBDseq_Analysis.ipynb’\n",
      "\n",
      "owl.fish.washington 100%[===================>] 412.88K  --.-KB/s    in 0.05s   \n",
      "\n",
      "2016-11-17 22:56:04 (8.43 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Oly_MBDseq_Analysis.ipynb’ saved [422787/422787]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/icons/text.gif\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 229 [image/gif]\n",
      "Saving to: ‘owl.fish.washington.edu/icons/text.gif’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     229  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (27.3 MB/s) - ‘owl.fish.washington.edu/icons/text.gif’ saved [229/229]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/readme.md\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 0 [text/plain]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/readme.md’\n",
      "\n",
      "owl.fish.washington     [ <=>                ]       0  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (0.00 B/s) - ‘owl.fish.washington.edu/symbiodinium/readme.md’ saved [0/0]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/?C=N;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1526 (1.5K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/index.html?C=N;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.49K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (104 MB/s) - ‘owl.fish.washington.edu/symbiodinium/index.html?C=N;O=A’ saved [1526/1526]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/?C=M;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1526 (1.5K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/index.html?C=M;O=D’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.49K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (162 MB/s) - ‘owl.fish.washington.edu/symbiodinium/index.html?C=M;O=D’ saved [1526/1526]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/?C=S;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1526 (1.5K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/index.html?C=S;O=D’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.49K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:04 (162 MB/s) - ‘owl.fish.washington.edu/symbiodinium/index.html?C=S;O=D’ saved [1526/1526]\n",
      "\n",
      "--2016-11-17 22:56:04--  http://owl.fish.washington.edu/symbiodinium/?C=D;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1526 (1.5K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/index.html?C=D;O=D’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.49K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (146 MB/s) - ‘owl.fish.washington.edu/symbiodinium/index.html?C=D;O=D’ saved [1526/1526]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/?C=N;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 924 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=N;O=D’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     924  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (97.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=N;O=D’ saved [924/924]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/?C=M;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 924 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=M;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     924  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (97.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=M;O=A’ saved [924/924]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/?C=S;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 924 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=S;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     924  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (88.1 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=S;O=A’ saved [924/924]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/?C=D;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 924 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=D;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     924  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (97.9 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=D;O=A’ saved [924/924]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1409 (1.4K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/index.html’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.38K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (149 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/index.html’ saved [1409/1409]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/?C=N;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 924 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=N;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     924  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (110 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=N;O=A’ saved [924/924]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/?C=M;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 924 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=M;O=D’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     924  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (110 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=M;O=D’ saved [924/924]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/?C=S;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 924 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=S;O=D’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     924  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (88.1 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=S;O=D’ saved [924/924]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/?C=D;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 924 [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=D;O=D’\n",
      "\n",
      "owl.fish.washington 100%[===================>]     924  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (110 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/index.html?C=D;O=D’ saved [924/924]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/?C=N;O=D\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1409 (1.4K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/index.html?C=N;O=D’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.38K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (149 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/index.html?C=N;O=D’ saved [1409/1409]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/?C=M;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1409 (1.4K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/index.html?C=M;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.38K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (122 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/index.html?C=M;O=A’ saved [1409/1409]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/?C=S;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1409 (1.4K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/index.html?C=S;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.38K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (149 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/index.html?C=S;O=A’ saved [1409/1409]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/?C=D;O=A\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1409 (1.4K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/index.html?C=D;O=A’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.38K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (149 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/index.html?C=D;O=A’ saved [1409/1409]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/T4-1-40299735/\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1288 (1.3K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/T4-1-40299735/index.html’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.26K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (154 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/T4-1-40299735/index.html’ saved [1288/1288]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/T5-1-40303742/\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1288 (1.3K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/T5-1-40303742/index.html’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.26K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (136 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/T5-1-40303742/index.html’ saved [1288/1288]\n",
      "\n",
      "--2016-11-17 22:56:05--  http://owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/T7-1-40300760/\n",
      "Reusing existing connection to owl.fish.washington.edu:80.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 1288 (1.3K) [text/html]\n",
      "Saving to: ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/T7-1-40300760/index.html’\n",
      "\n",
      "owl.fish.washington 100%[===================>]   1.26K  --.-KB/s    in 0s      \n",
      "\n",
      "2016-11-17 22:56:05 (136 MB/s) - ‘owl.fish.washington.edu/symbiodinium/Mcap_Spawn/H_Putnam_RNA_run1-32840854/T7-1-40300760/index.html’ saved [1288/1288]\n",
      "\n",
      "FINISHED --2016-11-17 22:56:05--\n",
      "Total wall clock time: 22m 29s\n",
      "Downloaded: 162 files, 48G in 20m 48s (39.4 MB/s)\n"
     ]
    }
   ],
   "source": [
    "!wget -r http://owl.fish.washington.edu/symbiodinium/Oly_MBD/Data"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Move sequence and checksum data to Data file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "mv owl.fish.washington.edu/symbiodinium/Oly_MBD/Data/*.gz Data/MBD"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Cleanup index html files from wget of data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!rm -r owl.fish.washington.edu"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Generate checksum data for MBD files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!md5 Data/MBD/*R1.fastq.gz > Data/MBD/20160203_mbdseq_checksums_downloaded_files.md5"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Compare MBD checksum files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MD5 (Data/MBD/zr1394_10_s1_R1.fastq.gz) = a250c01814de9b454ede42fa9c091a1e\r\n",
      "MD5 (Data/MBD/zr1394_10_s2_R1.fastq.gz) = f9f6befdd02b3feade1cf0b9e449d418\r\n",
      "MD5 (Data/MBD/zr1394_10_s3_R1.fastq.gz) = b786135a75e0be89dc37fefc5624a96a\r\n",
      "MD5 (Data/MBD/zr1394_10_s4_R1.fastq.gz) = f053a45339c8d9a59dcc9bd349c18a14\r\n",
      "MD5 (Data/MBD/zr1394_10_s5_R1.fastq.gz) = 3a371a46a0083d663697833bda7e11de\r\n",
      "MD5 (Data/MBD/zr1394_10_s6_R1.fastq.gz) = 4de675460549b492e711edf07ca5aa8e\r\n",
      "MD5 (Data/MBD/zr1394_11_s1_R1.fastq.gz) = c8e030a4dd73f81077db5ae449ec7d7d\r\n",
      "MD5 (Data/MBD/zr1394_11_s2_R1.fastq.gz) = 1566c372dc875df0c9998da7ee77dc05\r\n",
      "MD5 (Data/MBD/zr1394_11_s3_R1.fastq.gz) = 8709e1f4c42b01569c3cbfd685030921\r\n",
      "MD5 (Data/MBD/zr1394_11_s4_R1.fastq.gz) = 1433a5ea9b285ad06c0feb673090e161\r\n",
      "MD5 (Data/MBD/zr1394_11_s5_R1.fastq.gz) = 843fead9eac2a04f3025c725899a2b5f\r\n",
      "MD5 (Data/MBD/zr1394_11_s6_R1.fastq.gz) = 562888576bf0687c7d1e0940e96fa017\r\n",
      "MD5 (Data/MBD/zr1394_12_s1_R1.fastq.gz) = f5a00f675825ad2243b0dbe32135706b\r\n",
      "MD5 (Data/MBD/zr1394_12_s2_R1.fastq.gz) = 00a41b1504ceeb65f8a995ed6ccc3b1f\r\n",
      "MD5 (Data/MBD/zr1394_12_s3_R1.fastq.gz) = 23efc3acf7d56b2a41d6ba7c70dbfcfc\r\n",
      "MD5 (Data/MBD/zr1394_12_s4_R1.fastq.gz) = a434f51317ab862a00ee3ed24270da5d\r\n",
      "MD5 (Data/MBD/zr1394_12_s5_R1.fastq.gz) = 83738c3dc688fb3434b20f08a66cadae\r\n",
      "MD5 (Data/MBD/zr1394_12_s6_R1.fastq.gz) = 324e77a110b71328fd63945192928c1f\r\n",
      "MD5 (Data/MBD/zr1394_13_s1_R1.fastq.gz) = 80364e2dbdfae42b50f898a79adf7398\r\n",
      "MD5 (Data/MBD/zr1394_13_s2_R1.fastq.gz) = d4e7862f224d8fa4da67e4b2c536186e\r\n",
      "MD5 (Data/MBD/zr1394_13_s3_R1.fastq.gz) = ba1e2aca7184cd10e3451b7e6e7aa756\r\n",
      "MD5 (Data/MBD/zr1394_13_s4_R1.fastq.gz) = c090164ece2adbad67be6057b44ffc74\r\n",
      "MD5 (Data/MBD/zr1394_13_s5_R1.fastq.gz) = 646ff22898b773a4f519ee62a5b5392d\r\n",
      "MD5 (Data/MBD/zr1394_13_s6_R1.fastq.gz) = 2be763c10189ca9279273249a46ba61a\r\n",
      "MD5 (Data/MBD/zr1394_14_s1_R1.fastq.gz) = 28a1c5018d36d853b57cabf62427058c\r\n",
      "MD5 (Data/MBD/zr1394_14_s2_R1.fastq.gz) = 1dd3887f0b1ec734e2899f46e383493b\r\n",
      "MD5 (Data/MBD/zr1394_14_s3_R1.fastq.gz) = 9bd70c7fe774455896ae1fb00422744f\r\n",
      "MD5 (Data/MBD/zr1394_14_s4_R1.fastq.gz) = 82903c6d945c0da5a4140bcd526ee639\r\n",
      "MD5 (Data/MBD/zr1394_14_s5_R1.fastq.gz) = 63d3f95f4ae025f56c4ce41293a507ec\r\n",
      "MD5 (Data/MBD/zr1394_14_s6_R1.fastq.gz) = 7655a632d2bb3479d9b2add01678d9b3\r\n",
      "MD5 (Data/MBD/zr1394_15_s1_R1.fastq.gz) = 343a10a62d399151611332651853e197\r\n",
      "MD5 (Data/MBD/zr1394_15_s2_R1.fastq.gz) = ed27f0aa7a2e3e992bf4160402cd2c1c\r\n",
      "MD5 (Data/MBD/zr1394_15_s3_R1.fastq.gz) = a9d0a7489fd37c8c88397f6cd6bf2f09\r\n",
      "MD5 (Data/MBD/zr1394_15_s4_R1.fastq.gz) = 21204f0c8be5ab39246e3e43104f227f\r\n",
      "MD5 (Data/MBD/zr1394_15_s5_R1.fastq.gz) = dcd5145ff1da2352708d976452847924\r\n",
      "MD5 (Data/MBD/zr1394_15_s6_R1.fastq.gz) = 877527c7419d58dfa6670f0751c1c9b9\r\n",
      "MD5 (Data/MBD/zr1394_16_s1_R1.fastq.gz) = 7439246545c3793ac9ef028659125f4e\r\n",
      "MD5 (Data/MBD/zr1394_16_s2_R1.fastq.gz) = 923df16b61ab02dc1e11ff6b6aca9e09\r\n",
      "MD5 (Data/MBD/zr1394_16_s3_R1.fastq.gz) = aaba83078466b928f8518da1f1f156ad\r\n",
      "MD5 (Data/MBD/zr1394_16_s4_R1.fastq.gz) = 19573ffc1b6d7bd852945f28caecec58\r\n",
      "MD5 (Data/MBD/zr1394_16_s5_R1.fastq.gz) = 0f2e375f392f00b84f3258ae34509816\r\n",
      "MD5 (Data/MBD/zr1394_16_s6_R1.fastq.gz) = 71de5b5631b3570b8b8f28cc5d898fb1\r\n",
      "MD5 (Data/MBD/zr1394_17_s1_R1.fastq.gz) = 2261029b9675bb78c9c6282c9c8744fe\r\n",
      "MD5 (Data/MBD/zr1394_17_s2_R1.fastq.gz) = 1d1cd5510690bb0540767f13924d4fc2\r\n",
      "MD5 (Data/MBD/zr1394_17_s3_R1.fastq.gz) = 38d11d11dfca0ed7ec862c7323f5b697\r\n",
      "MD5 (Data/MBD/zr1394_17_s4_R1.fastq.gz) = dd7e0e4835672f92379c002e828f0ea2\r\n",
      "MD5 (Data/MBD/zr1394_17_s5_R1.fastq.gz) = dba14a0d79abb5fbd49b3786ee0bf8ec\r\n",
      "MD5 (Data/MBD/zr1394_17_s6_R1.fastq.gz) = 4046c422b3346b4defb9bd3bee964842\r\n",
      "MD5 (Data/MBD/zr1394_18_s1_R1.fastq.gz) = f4000ee31153e49d6438d748ed5a79f1\r\n",
      "MD5 (Data/MBD/zr1394_18_s2_R1.fastq.gz) = 235bddb23580ac89ac6b06f46fc548d7\r\n",
      "MD5 (Data/MBD/zr1394_18_s3_R1.fastq.gz) = c86592e818a80b7c82904c9949c351a6\r\n",
      "MD5 (Data/MBD/zr1394_18_s4_R1.fastq.gz) = dda4ccca18a906a17ff259588e414a3b\r\n",
      "MD5 (Data/MBD/zr1394_18_s5_R1.fastq.gz) = 38eb2b2fe2af9b3857aa43fca2ab11ba\r\n",
      "MD5 (Data/MBD/zr1394_18_s6_R1.fastq.gz) = 1fddd3b68d0cb7bae94e6e111ae449bf\r\n",
      "MD5 (Data/MBD/zr1394_1_s1_R1.fastq.gz) = 671f2e339aa3030d7f9e646643cc08c2\r\n",
      "MD5 (Data/MBD/zr1394_1_s2_R1.fastq.gz) = 2cdce5ac769bd6b0bedf2c1cb2ecd3c4\r\n",
      "MD5 (Data/MBD/zr1394_1_s3_R1.fastq.gz) = e5f1fbf496f17ab9b611f1f8272bbf78\r\n",
      "MD5 (Data/MBD/zr1394_1_s4_R1.fastq.gz) = 56d4194b09c85831a1cdf05f2322d883\r\n",
      "MD5 (Data/MBD/zr1394_1_s5_R1.fastq.gz) = 4fda8fa074b543ba01ccf41515aae53c\r\n",
      "MD5 (Data/MBD/zr1394_1_s6_R1.fastq.gz) = 3e458e6ca9e757bedc3ba0edc9d96260\r\n",
      "MD5 (Data/MBD/zr1394_2_s1_R1.fastq.gz) = 297f4a4636374740693d636410d29c0d\r\n",
      "MD5 (Data/MBD/zr1394_2_s2_R1.fastq.gz) = db1b4aedee88723df1b480c5b958a0b9\r\n",
      "MD5 (Data/MBD/zr1394_2_s3_R1.fastq.gz) = 5c449a59970b09675754db7e58e1f05a\r\n",
      "MD5 (Data/MBD/zr1394_2_s4_R1.fastq.gz) = 51300018d3f042c1a098bbe73d87b67d\r\n",
      "MD5 (Data/MBD/zr1394_2_s5_R1.fastq.gz) = a27d74c17aa1cb31a960ec84cb3213f4\r\n",
      "MD5 (Data/MBD/zr1394_2_s6_R1.fastq.gz) = 6af578b185c9db872331877750e97b02\r\n",
      "MD5 (Data/MBD/zr1394_3_s1_R1.fastq.gz) = e4f5c8f68a67ea9a999c28f2e42d8e6c\r\n",
      "MD5 (Data/MBD/zr1394_3_s2_R1.fastq.gz) = ea190a6b473a6f5260eca8cc0b401096\r\n",
      "MD5 (Data/MBD/zr1394_3_s3_R1.fastq.gz) = 4d8769496b64b12f237a0a7f982f91d5\r\n",
      "MD5 (Data/MBD/zr1394_3_s4_R1.fastq.gz) = a40b9b2b2e404e4eaac5cd62712858de\r\n",
      "MD5 (Data/MBD/zr1394_3_s5_R1.fastq.gz) = 883fce829a8f8c692a6903669220519c\r\n",
      "MD5 (Data/MBD/zr1394_3_s6_R1.fastq.gz) = 4412cc2690c5c2215f9c74e12cfd31d1\r\n",
      "MD5 (Data/MBD/zr1394_4_s1_R1.fastq.gz) = 54e9e661735e8f31c7b9b1bb3d4eb355\r\n",
      "MD5 (Data/MBD/zr1394_4_s2_R1.fastq.gz) = 4281597fa351e6e9c8f0d2ac2c38d0ec\r\n",
      "MD5 (Data/MBD/zr1394_4_s3_R1.fastq.gz) = eb6ff666b8ba721e7c5f82b2b68ef52b\r\n",
      "MD5 (Data/MBD/zr1394_4_s4_R1.fastq.gz) = 4eb1b68ab170e753a0cf4c2145ee6db0\r\n",
      "MD5 (Data/MBD/zr1394_4_s5_R1.fastq.gz) = 5ec140353392fe1de7397c239fc78e56\r\n",
      "MD5 (Data/MBD/zr1394_4_s6_R1.fastq.gz) = c28d4844dd701efa0f4a93a83c0f12f5\r\n",
      "MD5 (Data/MBD/zr1394_5_s1_R1.fastq.gz) = 007aff8f0a287de16299b394e671f5a3\r\n",
      "MD5 (Data/MBD/zr1394_5_s2_R1.fastq.gz) = 2bd24b92b1fad22552ad2f638e95de8c\r\n",
      "MD5 (Data/MBD/zr1394_5_s3_R1.fastq.gz) = b00ac358ad56605580265099076f480e\r\n",
      "MD5 (Data/MBD/zr1394_5_s4_R1.fastq.gz) = ebaad09c2db7166ae9e596f35d302b31\r\n",
      "MD5 (Data/MBD/zr1394_5_s5_R1.fastq.gz) = bec03cfacc459aa73798b8383a8cd2a9\r\n",
      "MD5 (Data/MBD/zr1394_5_s6_R1.fastq.gz) = 58162f7e654529bd6bb64846b5b41955\r\n",
      "MD5 (Data/MBD/zr1394_6_s1_R1.fastq.gz) = 32c2fff961f6a7ecaa17c364b274237f\r\n",
      "MD5 (Data/MBD/zr1394_6_s2_R1.fastq.gz) = 1eb423573b96f9545c143f3ca9b54a2a\r\n",
      "MD5 (Data/MBD/zr1394_6_s3_R1.fastq.gz) = 34785d0abec0b6f5b07c02a05e49d371\r\n",
      "MD5 (Data/MBD/zr1394_6_s4_R1.fastq.gz) = 3c2950ef9f2a1d75fa327ca2d0352ab8\r\n",
      "MD5 (Data/MBD/zr1394_6_s5_R1.fastq.gz) = a4ff9322d7fb4dcd2bf41ce74d74b4e5\r\n",
      "MD5 (Data/MBD/zr1394_6_s6_R1.fastq.gz) = 526dfad8294a330a444bf60e06bdc622\r\n",
      "MD5 (Data/MBD/zr1394_7_s1_R1.fastq.gz) = 2540597fd38a7f768d3c5576b0c51589\r\n",
      "MD5 (Data/MBD/zr1394_7_s2_R1.fastq.gz) = 9aca4d83d2efd8611fc8c2b15f9258c0\r\n",
      "MD5 (Data/MBD/zr1394_7_s3_R1.fastq.gz) = 123458c626a77ba4518f4bbe78125dad\r\n",
      "MD5 (Data/MBD/zr1394_7_s4_R1.fastq.gz) = 4e50b279d19d5fb76aa61e9a38dc5ec1\r\n",
      "MD5 (Data/MBD/zr1394_7_s5_R1.fastq.gz) = 5957bdc46aaae1731ee79dddc99c15c5\r\n",
      "MD5 (Data/MBD/zr1394_7_s6_R1.fastq.gz) = e496cc8c4bb18278c77464d45a3d49c4\r\n",
      "MD5 (Data/MBD/zr1394_8_s1_R1.fastq.gz) = f796e1d710d19399ea646948e64f881d\r\n",
      "MD5 (Data/MBD/zr1394_8_s2_R1.fastq.gz) = b9b3dcf27ccde3a54148daf04bbddcfd\r\n",
      "MD5 (Data/MBD/zr1394_8_s3_R1.fastq.gz) = 3fa868436c83cfea119704307f3fe11c\r\n",
      "MD5 (Data/MBD/zr1394_8_s4_R1.fastq.gz) = e5550ee2856d509e31e66f672abcb641\r\n",
      "MD5 (Data/MBD/zr1394_8_s5_R1.fastq.gz) = a1d3225bef6c126293e45298f578e6b2\r\n",
      "MD5 (Data/MBD/zr1394_8_s6_R1.fastq.gz) = 9fb727b5e221ce1cd4cfd75e46cf30cf\r\n",
      "MD5 (Data/MBD/zr1394_9_s1_R1.fastq.gz) = d3b51f8451199c3e39cdac8d015cf989\r\n",
      "MD5 (Data/MBD/zr1394_9_s2_R1.fastq.gz) = 004d8eb6231e7c32e5d6de4c78c28af6\r\n",
      "MD5 (Data/MBD/zr1394_9_s3_R1.fastq.gz) = d052ee9b6c88f9cd5d38bb2ad73cc118\r\n",
      "MD5 (Data/MBD/zr1394_9_s4_R1.fastq.gz) = ed793f38ee309ab252a5a2e6b3c7d3d7\r\n",
      "MD5 (Data/MBD/zr1394_9_s5_R1.fastq.gz) = f6092513aa5d7811d4207228bd5336e7\r\n",
      "MD5 (Data/MBD/zr1394_9_s6_R1.fastq.gz) = 5b0f480871406bbb5341bfcd2d95cec9\r\n"
     ]
    }
   ],
   "source": [
    "!cat Data/MBD/20160203_mbdseq_checksums_downloaded_files.md5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!gunzip Data/MBD/checksums.md5.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "mv Data/MBD/checksums.md5 Data/MBD/20160203_mbdseq_checksums.md5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MD5 (zr1394_10_s1_R1.fastq.gz) = a250c01814de9b454ede42fa9c091a1e\r\n",
      "MD5 (zr1394_10_s2_R1.fastq.gz) = f9f6befdd02b3feade1cf0b9e449d418\r\n",
      "MD5 (zr1394_10_s3_R1.fastq.gz) = b786135a75e0be89dc37fefc5624a96a\r\n",
      "MD5 (zr1394_10_s4_R1.fastq.gz) = f053a45339c8d9a59dcc9bd349c18a14\r\n",
      "MD5 (zr1394_10_s5_R1.fastq.gz) = 3a371a46a0083d663697833bda7e11de\r\n",
      "MD5 (zr1394_10_s6_R1.fastq.gz) = 4de675460549b492e711edf07ca5aa8e\r\n",
      "MD5 (zr1394_11_s1_R1.fastq.gz) = c8e030a4dd73f81077db5ae449ec7d7d\r\n",
      "MD5 (zr1394_11_s2_R1.fastq.gz) = 1566c372dc875df0c9998da7ee77dc05\r\n",
      "MD5 (zr1394_11_s3_R1.fastq.gz) = 8709e1f4c42b01569c3cbfd685030921\r\n",
      "MD5 (zr1394_11_s4_R1.fastq.gz) = 1433a5ea9b285ad06c0feb673090e161\r\n",
      "MD5 (zr1394_11_s5_R1.fastq.gz) = 843fead9eac2a04f3025c725899a2b5f\r\n",
      "MD5 (zr1394_11_s6_R1.fastq.gz) = 562888576bf0687c7d1e0940e96fa017\r\n",
      "MD5 (zr1394_12_s1_R1.fastq.gz) = f5a00f675825ad2243b0dbe32135706b\r\n",
      "MD5 (zr1394_12_s2_R1.fastq.gz) = 00a41b1504ceeb65f8a995ed6ccc3b1f\r\n",
      "MD5 (zr1394_12_s3_R1.fastq.gz) = 23efc3acf7d56b2a41d6ba7c70dbfcfc\r\n",
      "MD5 (zr1394_12_s4_R1.fastq.gz) = a434f51317ab862a00ee3ed24270da5d\r\n",
      "MD5 (zr1394_12_s5_R1.fastq.gz) = 83738c3dc688fb3434b20f08a66cadae\r\n",
      "MD5 (zr1394_12_s6_R1.fastq.gz) = 324e77a110b71328fd63945192928c1f\r\n",
      "MD5 (zr1394_13_s1_R1.fastq.gz) = 80364e2dbdfae42b50f898a79adf7398\r\n",
      "MD5 (zr1394_13_s2_R1.fastq.gz) = d4e7862f224d8fa4da67e4b2c536186e\r\n",
      "MD5 (zr1394_13_s3_R1.fastq.gz) = ba1e2aca7184cd10e3451b7e6e7aa756\r\n",
      "MD5 (zr1394_13_s4_R1.fastq.gz) = c090164ece2adbad67be6057b44ffc74\r\n",
      "MD5 (zr1394_13_s5_R1.fastq.gz) = 646ff22898b773a4f519ee62a5b5392d\r\n",
      "MD5 (zr1394_13_s6_R1.fastq.gz) = 2be763c10189ca9279273249a46ba61a\r\n",
      "MD5 (zr1394_14_s1_R1.fastq.gz) = 28a1c5018d36d853b57cabf62427058c\r\n",
      "MD5 (zr1394_14_s2_R1.fastq.gz) = 1dd3887f0b1ec734e2899f46e383493b\r\n",
      "MD5 (zr1394_14_s3_R1.fastq.gz) = 9bd70c7fe774455896ae1fb00422744f\r\n",
      "MD5 (zr1394_14_s4_R1.fastq.gz) = 82903c6d945c0da5a4140bcd526ee639\r\n",
      "MD5 (zr1394_14_s5_R1.fastq.gz) = 63d3f95f4ae025f56c4ce41293a507ec\r\n",
      "MD5 (zr1394_14_s6_R1.fastq.gz) = 7655a632d2bb3479d9b2add01678d9b3\r\n",
      "MD5 (zr1394_15_s1_R1.fastq.gz) = 343a10a62d399151611332651853e197\r\n",
      "MD5 (zr1394_15_s2_R1.fastq.gz) = ed27f0aa7a2e3e992bf4160402cd2c1c\r\n",
      "MD5 (zr1394_15_s3_R1.fastq.gz) = a9d0a7489fd37c8c88397f6cd6bf2f09\r\n",
      "MD5 (zr1394_15_s4_R1.fastq.gz) = 21204f0c8be5ab39246e3e43104f227f\r\n",
      "MD5 (zr1394_15_s5_R1.fastq.gz) = dcd5145ff1da2352708d976452847924\r\n",
      "MD5 (zr1394_15_s6_R1.fastq.gz) = 877527c7419d58dfa6670f0751c1c9b9\r\n",
      "MD5 (zr1394_16_s1_R1.fastq.gz) = 7439246545c3793ac9ef028659125f4e\r\n",
      "MD5 (zr1394_16_s2_R1.fastq.gz) = 923df16b61ab02dc1e11ff6b6aca9e09\r\n",
      "MD5 (zr1394_16_s3_R1.fastq.gz) = aaba83078466b928f8518da1f1f156ad\r\n",
      "MD5 (zr1394_16_s4_R1.fastq.gz) = 19573ffc1b6d7bd852945f28caecec58\r\n",
      "MD5 (zr1394_16_s5_R1.fastq.gz) = 0f2e375f392f00b84f3258ae34509816\r\n",
      "MD5 (zr1394_16_s6_R1.fastq.gz) = 71de5b5631b3570b8b8f28cc5d898fb1\r\n",
      "MD5 (zr1394_17_s1_R1.fastq.gz) = 2261029b9675bb78c9c6282c9c8744fe\r\n",
      "MD5 (zr1394_17_s2_R1.fastq.gz) = 1d1cd5510690bb0540767f13924d4fc2\r\n",
      "MD5 (zr1394_17_s3_R1.fastq.gz) = 38d11d11dfca0ed7ec862c7323f5b697\r\n",
      "MD5 (zr1394_17_s4_R1.fastq.gz) = dd7e0e4835672f92379c002e828f0ea2\r\n",
      "MD5 (zr1394_17_s5_R1.fastq.gz) = dba14a0d79abb5fbd49b3786ee0bf8ec\r\n",
      "MD5 (zr1394_17_s6_R1.fastq.gz) = 4046c422b3346b4defb9bd3bee964842\r\n",
      "MD5 (zr1394_18_s1_R1.fastq.gz) = f4000ee31153e49d6438d748ed5a79f1\r\n",
      "MD5 (zr1394_18_s2_R1.fastq.gz) = 235bddb23580ac89ac6b06f46fc548d7\r\n",
      "MD5 (zr1394_18_s3_R1.fastq.gz) = c86592e818a80b7c82904c9949c351a6\r\n",
      "MD5 (zr1394_18_s4_R1.fastq.gz) = dda4ccca18a906a17ff259588e414a3b\r\n",
      "MD5 (zr1394_18_s5_R1.fastq.gz) = 38eb2b2fe2af9b3857aa43fca2ab11ba\r\n",
      "MD5 (zr1394_18_s6_R1.fastq.gz) = 1fddd3b68d0cb7bae94e6e111ae449bf\r\n",
      "MD5 (zr1394_1_s1_R1.fastq.gz) = 671f2e339aa3030d7f9e646643cc08c2\r\n",
      "MD5 (zr1394_1_s2_R1.fastq.gz) = 2cdce5ac769bd6b0bedf2c1cb2ecd3c4\r\n",
      "MD5 (zr1394_1_s3_R1.fastq.gz) = e5f1fbf496f17ab9b611f1f8272bbf78\r\n",
      "MD5 (zr1394_1_s4_R1.fastq.gz) = 56d4194b09c85831a1cdf05f2322d883\r\n",
      "MD5 (zr1394_1_s5_R1.fastq.gz) = 4fda8fa074b543ba01ccf41515aae53c\r\n",
      "MD5 (zr1394_1_s6_R1.fastq.gz) = 3e458e6ca9e757bedc3ba0edc9d96260\r\n",
      "MD5 (zr1394_2_s1_R1.fastq.gz) = 297f4a4636374740693d636410d29c0d\r\n",
      "MD5 (zr1394_2_s2_R1.fastq.gz) = db1b4aedee88723df1b480c5b958a0b9\r\n",
      "MD5 (zr1394_2_s3_R1.fastq.gz) = 5c449a59970b09675754db7e58e1f05a\r\n",
      "MD5 (zr1394_2_s4_R1.fastq.gz) = 51300018d3f042c1a098bbe73d87b67d\r\n",
      "MD5 (zr1394_2_s5_R1.fastq.gz) = a27d74c17aa1cb31a960ec84cb3213f4\r\n",
      "MD5 (zr1394_2_s6_R1.fastq.gz) = 6af578b185c9db872331877750e97b02\r\n",
      "MD5 (zr1394_3_s1_R1.fastq.gz) = e4f5c8f68a67ea9a999c28f2e42d8e6c\r\n",
      "MD5 (zr1394_3_s2_R1.fastq.gz) = ea190a6b473a6f5260eca8cc0b401096\r\n",
      "MD5 (zr1394_3_s3_R1.fastq.gz) = 4d8769496b64b12f237a0a7f982f91d5\r\n",
      "MD5 (zr1394_3_s4_R1.fastq.gz) = a40b9b2b2e404e4eaac5cd62712858de\r\n",
      "MD5 (zr1394_3_s5_R1.fastq.gz) = 883fce829a8f8c692a6903669220519c\r\n",
      "MD5 (zr1394_3_s6_R1.fastq.gz) = 4412cc2690c5c2215f9c74e12cfd31d1\r\n",
      "MD5 (zr1394_4_s1_R1.fastq.gz) = 54e9e661735e8f31c7b9b1bb3d4eb355\r\n",
      "MD5 (zr1394_4_s2_R1.fastq.gz) = 4281597fa351e6e9c8f0d2ac2c38d0ec\r\n",
      "MD5 (zr1394_4_s3_R1.fastq.gz) = eb6ff666b8ba721e7c5f82b2b68ef52b\r\n",
      "MD5 (zr1394_4_s4_R1.fastq.gz) = 4eb1b68ab170e753a0cf4c2145ee6db0\r\n",
      "MD5 (zr1394_4_s5_R1.fastq.gz) = 5ec140353392fe1de7397c239fc78e56\r\n",
      "MD5 (zr1394_4_s6_R1.fastq.gz) = c28d4844dd701efa0f4a93a83c0f12f5\r\n",
      "MD5 (zr1394_5_s1_R1.fastq.gz) = 007aff8f0a287de16299b394e671f5a3\r\n",
      "MD5 (zr1394_5_s2_R1.fastq.gz) = 2bd24b92b1fad22552ad2f638e95de8c\r\n",
      "MD5 (zr1394_5_s3_R1.fastq.gz) = b00ac358ad56605580265099076f480e\r\n",
      "MD5 (zr1394_5_s4_R1.fastq.gz) = ebaad09c2db7166ae9e596f35d302b31\r\n",
      "MD5 (zr1394_5_s5_R1.fastq.gz) = bec03cfacc459aa73798b8383a8cd2a9\r\n",
      "MD5 (zr1394_5_s6_R1.fastq.gz) = 58162f7e654529bd6bb64846b5b41955\r\n",
      "MD5 (zr1394_6_s1_R1.fastq.gz) = 32c2fff961f6a7ecaa17c364b274237f\r\n",
      "MD5 (zr1394_6_s2_R1.fastq.gz) = 1eb423573b96f9545c143f3ca9b54a2a\r\n",
      "MD5 (zr1394_6_s3_R1.fastq.gz) = 34785d0abec0b6f5b07c02a05e49d371\r\n",
      "MD5 (zr1394_6_s4_R1.fastq.gz) = 3c2950ef9f2a1d75fa327ca2d0352ab8\r\n",
      "MD5 (zr1394_6_s5_R1.fastq.gz) = a4ff9322d7fb4dcd2bf41ce74d74b4e5\r\n",
      "MD5 (zr1394_6_s6_R1.fastq.gz) = 526dfad8294a330a444bf60e06bdc622\r\n",
      "MD5 (zr1394_7_s1_R1.fastq.gz) = 2540597fd38a7f768d3c5576b0c51589\r\n",
      "MD5 (zr1394_7_s2_R1.fastq.gz) = 9aca4d83d2efd8611fc8c2b15f9258c0\r\n",
      "MD5 (zr1394_7_s3_R1.fastq.gz) = 123458c626a77ba4518f4bbe78125dad\r\n",
      "MD5 (zr1394_7_s4_R1.fastq.gz) = 4e50b279d19d5fb76aa61e9a38dc5ec1\r\n",
      "MD5 (zr1394_7_s5_R1.fastq.gz) = 5957bdc46aaae1731ee79dddc99c15c5\r\n",
      "MD5 (zr1394_7_s6_R1.fastq.gz) = e496cc8c4bb18278c77464d45a3d49c4\r\n",
      "MD5 (zr1394_8_s1_R1.fastq.gz) = f796e1d710d19399ea646948e64f881d\r\n",
      "MD5 (zr1394_8_s2_R1.fastq.gz) = b9b3dcf27ccde3a54148daf04bbddcfd\r\n",
      "MD5 (zr1394_8_s3_R1.fastq.gz) = 3fa868436c83cfea119704307f3fe11c\r\n",
      "MD5 (zr1394_8_s4_R1.fastq.gz) = e5550ee2856d509e31e66f672abcb641\r\n",
      "MD5 (zr1394_8_s5_R1.fastq.gz) = a1d3225bef6c126293e45298f578e6b2\r\n",
      "MD5 (zr1394_8_s6_R1.fastq.gz) = 9fb727b5e221ce1cd4cfd75e46cf30cf\r\n",
      "MD5 (zr1394_9_s1_R1.fastq.gz) = d3b51f8451199c3e39cdac8d015cf989\r\n",
      "MD5 (zr1394_9_s2_R1.fastq.gz) = 004d8eb6231e7c32e5d6de4c78c28af6\r\n",
      "MD5 (zr1394_9_s3_R1.fastq.gz) = d052ee9b6c88f9cd5d38bb2ad73cc118\r\n",
      "MD5 (zr1394_9_s4_R1.fastq.gz) = ed793f38ee309ab252a5a2e6b3c7d3d7\r\n",
      "MD5 (zr1394_9_s5_R1.fastq.gz) = f6092513aa5d7811d4207228bd5336e7\r\n",
      "MD5 (zr1394_9_s6_R1.fastq.gz) = 5b0f480871406bbb5341bfcd2d95cec9\r\n"
     ]
    }
   ],
   "source": [
    "!cat Data/MBD/20160203_mbdseq_checksums.md5"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* compare checksum files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!paste Data/MBD/20160203_mbdseq_checksums.md5 Data/MBD/20160203_mbdseq_checksums_downloaded_files.md5 | cut -f 1,2,3 > Data/MBD/checksums_comparison.md5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MD5 (zr1394_10_s1_R1.fastq.gz) = a250c01814de9b454ede42fa9c091a1e\tMD5 (Data/MBD/zr1394_10_s1_R1.fastq.gz) = a250c01814de9b454ede42fa9c091a1e\r\n",
      "MD5 (zr1394_10_s2_R1.fastq.gz) = f9f6befdd02b3feade1cf0b9e449d418\tMD5 (Data/MBD/zr1394_10_s2_R1.fastq.gz) = f9f6befdd02b3feade1cf0b9e449d418\r\n",
      "MD5 (zr1394_10_s3_R1.fastq.gz) = b786135a75e0be89dc37fefc5624a96a\tMD5 (Data/MBD/zr1394_10_s3_R1.fastq.gz) = b786135a75e0be89dc37fefc5624a96a\r\n",
      "MD5 (zr1394_10_s4_R1.fastq.gz) = f053a45339c8d9a59dcc9bd349c18a14\tMD5 (Data/MBD/zr1394_10_s4_R1.fastq.gz) = f053a45339c8d9a59dcc9bd349c18a14\r\n",
      "MD5 (zr1394_10_s5_R1.fastq.gz) = 3a371a46a0083d663697833bda7e11de\tMD5 (Data/MBD/zr1394_10_s5_R1.fastq.gz) = 3a371a46a0083d663697833bda7e11de\r\n",
      "MD5 (zr1394_10_s6_R1.fastq.gz) = 4de675460549b492e711edf07ca5aa8e\tMD5 (Data/MBD/zr1394_10_s6_R1.fastq.gz) = 4de675460549b492e711edf07ca5aa8e\r\n",
      "MD5 (zr1394_11_s1_R1.fastq.gz) = c8e030a4dd73f81077db5ae449ec7d7d\tMD5 (Data/MBD/zr1394_11_s1_R1.fastq.gz) = c8e030a4dd73f81077db5ae449ec7d7d\r\n",
      "MD5 (zr1394_11_s2_R1.fastq.gz) = 1566c372dc875df0c9998da7ee77dc05\tMD5 (Data/MBD/zr1394_11_s2_R1.fastq.gz) = 1566c372dc875df0c9998da7ee77dc05\r\n",
      "MD5 (zr1394_11_s3_R1.fastq.gz) = 8709e1f4c42b01569c3cbfd685030921\tMD5 (Data/MBD/zr1394_11_s3_R1.fastq.gz) = 8709e1f4c42b01569c3cbfd685030921\r\n",
      "MD5 (zr1394_11_s4_R1.fastq.gz) = 1433a5ea9b285ad06c0feb673090e161\tMD5 (Data/MBD/zr1394_11_s4_R1.fastq.gz) = 1433a5ea9b285ad06c0feb673090e161\r\n",
      "MD5 (zr1394_11_s5_R1.fastq.gz) = 843fead9eac2a04f3025c725899a2b5f\tMD5 (Data/MBD/zr1394_11_s5_R1.fastq.gz) = 843fead9eac2a04f3025c725899a2b5f\r\n",
      "MD5 (zr1394_11_s6_R1.fastq.gz) = 562888576bf0687c7d1e0940e96fa017\tMD5 (Data/MBD/zr1394_11_s6_R1.fastq.gz) = 562888576bf0687c7d1e0940e96fa017\r\n",
      "MD5 (zr1394_12_s1_R1.fastq.gz) = f5a00f675825ad2243b0dbe32135706b\tMD5 (Data/MBD/zr1394_12_s1_R1.fastq.gz) = f5a00f675825ad2243b0dbe32135706b\r\n",
      "MD5 (zr1394_12_s2_R1.fastq.gz) = 00a41b1504ceeb65f8a995ed6ccc3b1f\tMD5 (Data/MBD/zr1394_12_s2_R1.fastq.gz) = 00a41b1504ceeb65f8a995ed6ccc3b1f\r\n",
      "MD5 (zr1394_12_s3_R1.fastq.gz) = 23efc3acf7d56b2a41d6ba7c70dbfcfc\tMD5 (Data/MBD/zr1394_12_s3_R1.fastq.gz) = 23efc3acf7d56b2a41d6ba7c70dbfcfc\r\n",
      "MD5 (zr1394_12_s4_R1.fastq.gz) = a434f51317ab862a00ee3ed24270da5d\tMD5 (Data/MBD/zr1394_12_s4_R1.fastq.gz) = a434f51317ab862a00ee3ed24270da5d\r\n",
      "MD5 (zr1394_12_s5_R1.fastq.gz) = 83738c3dc688fb3434b20f08a66cadae\tMD5 (Data/MBD/zr1394_12_s5_R1.fastq.gz) = 83738c3dc688fb3434b20f08a66cadae\r\n",
      "MD5 (zr1394_12_s6_R1.fastq.gz) = 324e77a110b71328fd63945192928c1f\tMD5 (Data/MBD/zr1394_12_s6_R1.fastq.gz) = 324e77a110b71328fd63945192928c1f\r\n",
      "MD5 (zr1394_13_s1_R1.fastq.gz) = 80364e2dbdfae42b50f898a79adf7398\tMD5 (Data/MBD/zr1394_13_s1_R1.fastq.gz) = 80364e2dbdfae42b50f898a79adf7398\r\n",
      "MD5 (zr1394_13_s2_R1.fastq.gz) = d4e7862f224d8fa4da67e4b2c536186e\tMD5 (Data/MBD/zr1394_13_s2_R1.fastq.gz) = d4e7862f224d8fa4da67e4b2c536186e\r\n",
      "MD5 (zr1394_13_s3_R1.fastq.gz) = ba1e2aca7184cd10e3451b7e6e7aa756\tMD5 (Data/MBD/zr1394_13_s3_R1.fastq.gz) = ba1e2aca7184cd10e3451b7e6e7aa756\r\n",
      "MD5 (zr1394_13_s4_R1.fastq.gz) = c090164ece2adbad67be6057b44ffc74\tMD5 (Data/MBD/zr1394_13_s4_R1.fastq.gz) = c090164ece2adbad67be6057b44ffc74\r\n",
      "MD5 (zr1394_13_s5_R1.fastq.gz) = 646ff22898b773a4f519ee62a5b5392d\tMD5 (Data/MBD/zr1394_13_s5_R1.fastq.gz) = 646ff22898b773a4f519ee62a5b5392d\r\n",
      "MD5 (zr1394_13_s6_R1.fastq.gz) = 2be763c10189ca9279273249a46ba61a\tMD5 (Data/MBD/zr1394_13_s6_R1.fastq.gz) = 2be763c10189ca9279273249a46ba61a\r\n",
      "MD5 (zr1394_14_s1_R1.fastq.gz) = 28a1c5018d36d853b57cabf62427058c\tMD5 (Data/MBD/zr1394_14_s1_R1.fastq.gz) = 28a1c5018d36d853b57cabf62427058c\r\n",
      "MD5 (zr1394_14_s2_R1.fastq.gz) = 1dd3887f0b1ec734e2899f46e383493b\tMD5 (Data/MBD/zr1394_14_s2_R1.fastq.gz) = 1dd3887f0b1ec734e2899f46e383493b\r\n",
      "MD5 (zr1394_14_s3_R1.fastq.gz) = 9bd70c7fe774455896ae1fb00422744f\tMD5 (Data/MBD/zr1394_14_s3_R1.fastq.gz) = 9bd70c7fe774455896ae1fb00422744f\r\n",
      "MD5 (zr1394_14_s4_R1.fastq.gz) = 82903c6d945c0da5a4140bcd526ee639\tMD5 (Data/MBD/zr1394_14_s4_R1.fastq.gz) = 82903c6d945c0da5a4140bcd526ee639\r\n",
      "MD5 (zr1394_14_s5_R1.fastq.gz) = 63d3f95f4ae025f56c4ce41293a507ec\tMD5 (Data/MBD/zr1394_14_s5_R1.fastq.gz) = 63d3f95f4ae025f56c4ce41293a507ec\r\n",
      "MD5 (zr1394_14_s6_R1.fastq.gz) = 7655a632d2bb3479d9b2add01678d9b3\tMD5 (Data/MBD/zr1394_14_s6_R1.fastq.gz) = 7655a632d2bb3479d9b2add01678d9b3\r\n",
      "MD5 (zr1394_15_s1_R1.fastq.gz) = 343a10a62d399151611332651853e197\tMD5 (Data/MBD/zr1394_15_s1_R1.fastq.gz) = 343a10a62d399151611332651853e197\r\n",
      "MD5 (zr1394_15_s2_R1.fastq.gz) = ed27f0aa7a2e3e992bf4160402cd2c1c\tMD5 (Data/MBD/zr1394_15_s2_R1.fastq.gz) = ed27f0aa7a2e3e992bf4160402cd2c1c\r\n",
      "MD5 (zr1394_15_s3_R1.fastq.gz) = a9d0a7489fd37c8c88397f6cd6bf2f09\tMD5 (Data/MBD/zr1394_15_s3_R1.fastq.gz) = a9d0a7489fd37c8c88397f6cd6bf2f09\r\n",
      "MD5 (zr1394_15_s4_R1.fastq.gz) = 21204f0c8be5ab39246e3e43104f227f\tMD5 (Data/MBD/zr1394_15_s4_R1.fastq.gz) = 21204f0c8be5ab39246e3e43104f227f\r\n",
      "MD5 (zr1394_15_s5_R1.fastq.gz) = dcd5145ff1da2352708d976452847924\tMD5 (Data/MBD/zr1394_15_s5_R1.fastq.gz) = dcd5145ff1da2352708d976452847924\r\n",
      "MD5 (zr1394_15_s6_R1.fastq.gz) = 877527c7419d58dfa6670f0751c1c9b9\tMD5 (Data/MBD/zr1394_15_s6_R1.fastq.gz) = 877527c7419d58dfa6670f0751c1c9b9\r\n",
      "MD5 (zr1394_16_s1_R1.fastq.gz) = 7439246545c3793ac9ef028659125f4e\tMD5 (Data/MBD/zr1394_16_s1_R1.fastq.gz) = 7439246545c3793ac9ef028659125f4e\r\n",
      "MD5 (zr1394_16_s2_R1.fastq.gz) = 923df16b61ab02dc1e11ff6b6aca9e09\tMD5 (Data/MBD/zr1394_16_s2_R1.fastq.gz) = 923df16b61ab02dc1e11ff6b6aca9e09\r\n",
      "MD5 (zr1394_16_s3_R1.fastq.gz) = aaba83078466b928f8518da1f1f156ad\tMD5 (Data/MBD/zr1394_16_s3_R1.fastq.gz) = aaba83078466b928f8518da1f1f156ad\r\n",
      "MD5 (zr1394_16_s4_R1.fastq.gz) = 19573ffc1b6d7bd852945f28caecec58\tMD5 (Data/MBD/zr1394_16_s4_R1.fastq.gz) = 19573ffc1b6d7bd852945f28caecec58\r\n",
      "MD5 (zr1394_16_s5_R1.fastq.gz) = 0f2e375f392f00b84f3258ae34509816\tMD5 (Data/MBD/zr1394_16_s5_R1.fastq.gz) = 0f2e375f392f00b84f3258ae34509816\r\n",
      "MD5 (zr1394_16_s6_R1.fastq.gz) = 71de5b5631b3570b8b8f28cc5d898fb1\tMD5 (Data/MBD/zr1394_16_s6_R1.fastq.gz) = 71de5b5631b3570b8b8f28cc5d898fb1\r\n",
      "MD5 (zr1394_17_s1_R1.fastq.gz) = 2261029b9675bb78c9c6282c9c8744fe\tMD5 (Data/MBD/zr1394_17_s1_R1.fastq.gz) = 2261029b9675bb78c9c6282c9c8744fe\r\n",
      "MD5 (zr1394_17_s2_R1.fastq.gz) = 1d1cd5510690bb0540767f13924d4fc2\tMD5 (Data/MBD/zr1394_17_s2_R1.fastq.gz) = 1d1cd5510690bb0540767f13924d4fc2\r\n",
      "MD5 (zr1394_17_s3_R1.fastq.gz) = 38d11d11dfca0ed7ec862c7323f5b697\tMD5 (Data/MBD/zr1394_17_s3_R1.fastq.gz) = 38d11d11dfca0ed7ec862c7323f5b697\r\n",
      "MD5 (zr1394_17_s4_R1.fastq.gz) = dd7e0e4835672f92379c002e828f0ea2\tMD5 (Data/MBD/zr1394_17_s4_R1.fastq.gz) = dd7e0e4835672f92379c002e828f0ea2\r\n",
      "MD5 (zr1394_17_s5_R1.fastq.gz) = dba14a0d79abb5fbd49b3786ee0bf8ec\tMD5 (Data/MBD/zr1394_17_s5_R1.fastq.gz) = dba14a0d79abb5fbd49b3786ee0bf8ec\r\n",
      "MD5 (zr1394_17_s6_R1.fastq.gz) = 4046c422b3346b4defb9bd3bee964842\tMD5 (Data/MBD/zr1394_17_s6_R1.fastq.gz) = 4046c422b3346b4defb9bd3bee964842\r\n",
      "MD5 (zr1394_18_s1_R1.fastq.gz) = f4000ee31153e49d6438d748ed5a79f1\tMD5 (Data/MBD/zr1394_18_s1_R1.fastq.gz) = f4000ee31153e49d6438d748ed5a79f1\r\n",
      "MD5 (zr1394_18_s2_R1.fastq.gz) = 235bddb23580ac89ac6b06f46fc548d7\tMD5 (Data/MBD/zr1394_18_s2_R1.fastq.gz) = 235bddb23580ac89ac6b06f46fc548d7\r\n",
      "MD5 (zr1394_18_s3_R1.fastq.gz) = c86592e818a80b7c82904c9949c351a6\tMD5 (Data/MBD/zr1394_18_s3_R1.fastq.gz) = c86592e818a80b7c82904c9949c351a6\r\n",
      "MD5 (zr1394_18_s4_R1.fastq.gz) = dda4ccca18a906a17ff259588e414a3b\tMD5 (Data/MBD/zr1394_18_s4_R1.fastq.gz) = dda4ccca18a906a17ff259588e414a3b\r\n",
      "MD5 (zr1394_18_s5_R1.fastq.gz) = 38eb2b2fe2af9b3857aa43fca2ab11ba\tMD5 (Data/MBD/zr1394_18_s5_R1.fastq.gz) = 38eb2b2fe2af9b3857aa43fca2ab11ba\r\n",
      "MD5 (zr1394_18_s6_R1.fastq.gz) = 1fddd3b68d0cb7bae94e6e111ae449bf\tMD5 (Data/MBD/zr1394_18_s6_R1.fastq.gz) = 1fddd3b68d0cb7bae94e6e111ae449bf\r\n",
      "MD5 (zr1394_1_s1_R1.fastq.gz) = 671f2e339aa3030d7f9e646643cc08c2\tMD5 (Data/MBD/zr1394_1_s1_R1.fastq.gz) = 671f2e339aa3030d7f9e646643cc08c2\r\n",
      "MD5 (zr1394_1_s2_R1.fastq.gz) = 2cdce5ac769bd6b0bedf2c1cb2ecd3c4\tMD5 (Data/MBD/zr1394_1_s2_R1.fastq.gz) = 2cdce5ac769bd6b0bedf2c1cb2ecd3c4\r\n",
      "MD5 (zr1394_1_s3_R1.fastq.gz) = e5f1fbf496f17ab9b611f1f8272bbf78\tMD5 (Data/MBD/zr1394_1_s3_R1.fastq.gz) = e5f1fbf496f17ab9b611f1f8272bbf78\r\n",
      "MD5 (zr1394_1_s4_R1.fastq.gz) = 56d4194b09c85831a1cdf05f2322d883\tMD5 (Data/MBD/zr1394_1_s4_R1.fastq.gz) = 56d4194b09c85831a1cdf05f2322d883\r\n",
      "MD5 (zr1394_1_s5_R1.fastq.gz) = 4fda8fa074b543ba01ccf41515aae53c\tMD5 (Data/MBD/zr1394_1_s5_R1.fastq.gz) = 4fda8fa074b543ba01ccf41515aae53c\r\n",
      "MD5 (zr1394_1_s6_R1.fastq.gz) = 3e458e6ca9e757bedc3ba0edc9d96260\tMD5 (Data/MBD/zr1394_1_s6_R1.fastq.gz) = 3e458e6ca9e757bedc3ba0edc9d96260\r\n",
      "MD5 (zr1394_2_s1_R1.fastq.gz) = 297f4a4636374740693d636410d29c0d\tMD5 (Data/MBD/zr1394_2_s1_R1.fastq.gz) = 297f4a4636374740693d636410d29c0d\r\n",
      "MD5 (zr1394_2_s2_R1.fastq.gz) = db1b4aedee88723df1b480c5b958a0b9\tMD5 (Data/MBD/zr1394_2_s2_R1.fastq.gz) = db1b4aedee88723df1b480c5b958a0b9\r\n",
      "MD5 (zr1394_2_s3_R1.fastq.gz) = 5c449a59970b09675754db7e58e1f05a\tMD5 (Data/MBD/zr1394_2_s3_R1.fastq.gz) = 5c449a59970b09675754db7e58e1f05a\r\n",
      "MD5 (zr1394_2_s4_R1.fastq.gz) = 51300018d3f042c1a098bbe73d87b67d\tMD5 (Data/MBD/zr1394_2_s4_R1.fastq.gz) = 51300018d3f042c1a098bbe73d87b67d\r\n",
      "MD5 (zr1394_2_s5_R1.fastq.gz) = a27d74c17aa1cb31a960ec84cb3213f4\tMD5 (Data/MBD/zr1394_2_s5_R1.fastq.gz) = a27d74c17aa1cb31a960ec84cb3213f4\r\n",
      "MD5 (zr1394_2_s6_R1.fastq.gz) = 6af578b185c9db872331877750e97b02\tMD5 (Data/MBD/zr1394_2_s6_R1.fastq.gz) = 6af578b185c9db872331877750e97b02\r\n",
      "MD5 (zr1394_3_s1_R1.fastq.gz) = e4f5c8f68a67ea9a999c28f2e42d8e6c\tMD5 (Data/MBD/zr1394_3_s1_R1.fastq.gz) = e4f5c8f68a67ea9a999c28f2e42d8e6c\r\n",
      "MD5 (zr1394_3_s2_R1.fastq.gz) = ea190a6b473a6f5260eca8cc0b401096\tMD5 (Data/MBD/zr1394_3_s2_R1.fastq.gz) = ea190a6b473a6f5260eca8cc0b401096\r\n",
      "MD5 (zr1394_3_s3_R1.fastq.gz) = 4d8769496b64b12f237a0a7f982f91d5\tMD5 (Data/MBD/zr1394_3_s3_R1.fastq.gz) = 4d8769496b64b12f237a0a7f982f91d5\r\n",
      "MD5 (zr1394_3_s4_R1.fastq.gz) = a40b9b2b2e404e4eaac5cd62712858de\tMD5 (Data/MBD/zr1394_3_s4_R1.fastq.gz) = a40b9b2b2e404e4eaac5cd62712858de\r\n",
      "MD5 (zr1394_3_s5_R1.fastq.gz) = 883fce829a8f8c692a6903669220519c\tMD5 (Data/MBD/zr1394_3_s5_R1.fastq.gz) = 883fce829a8f8c692a6903669220519c\r\n",
      "MD5 (zr1394_3_s6_R1.fastq.gz) = 4412cc2690c5c2215f9c74e12cfd31d1\tMD5 (Data/MBD/zr1394_3_s6_R1.fastq.gz) = 4412cc2690c5c2215f9c74e12cfd31d1\r\n",
      "MD5 (zr1394_4_s1_R1.fastq.gz) = 54e9e661735e8f31c7b9b1bb3d4eb355\tMD5 (Data/MBD/zr1394_4_s1_R1.fastq.gz) = 54e9e661735e8f31c7b9b1bb3d4eb355\r\n",
      "MD5 (zr1394_4_s2_R1.fastq.gz) = 4281597fa351e6e9c8f0d2ac2c38d0ec\tMD5 (Data/MBD/zr1394_4_s2_R1.fastq.gz) = 4281597fa351e6e9c8f0d2ac2c38d0ec\r\n",
      "MD5 (zr1394_4_s3_R1.fastq.gz) = eb6ff666b8ba721e7c5f82b2b68ef52b\tMD5 (Data/MBD/zr1394_4_s3_R1.fastq.gz) = eb6ff666b8ba721e7c5f82b2b68ef52b\r\n",
      "MD5 (zr1394_4_s4_R1.fastq.gz) = 4eb1b68ab170e753a0cf4c2145ee6db0\tMD5 (Data/MBD/zr1394_4_s4_R1.fastq.gz) = 4eb1b68ab170e753a0cf4c2145ee6db0\r\n",
      "MD5 (zr1394_4_s5_R1.fastq.gz) = 5ec140353392fe1de7397c239fc78e56\tMD5 (Data/MBD/zr1394_4_s5_R1.fastq.gz) = 5ec140353392fe1de7397c239fc78e56\r\n",
      "MD5 (zr1394_4_s6_R1.fastq.gz) = c28d4844dd701efa0f4a93a83c0f12f5\tMD5 (Data/MBD/zr1394_4_s6_R1.fastq.gz) = c28d4844dd701efa0f4a93a83c0f12f5\r\n",
      "MD5 (zr1394_5_s1_R1.fastq.gz) = 007aff8f0a287de16299b394e671f5a3\tMD5 (Data/MBD/zr1394_5_s1_R1.fastq.gz) = 007aff8f0a287de16299b394e671f5a3\r\n",
      "MD5 (zr1394_5_s2_R1.fastq.gz) = 2bd24b92b1fad22552ad2f638e95de8c\tMD5 (Data/MBD/zr1394_5_s2_R1.fastq.gz) = 2bd24b92b1fad22552ad2f638e95de8c\r\n",
      "MD5 (zr1394_5_s3_R1.fastq.gz) = b00ac358ad56605580265099076f480e\tMD5 (Data/MBD/zr1394_5_s3_R1.fastq.gz) = b00ac358ad56605580265099076f480e\r\n",
      "MD5 (zr1394_5_s4_R1.fastq.gz) = ebaad09c2db7166ae9e596f35d302b31\tMD5 (Data/MBD/zr1394_5_s4_R1.fastq.gz) = ebaad09c2db7166ae9e596f35d302b31\r\n",
      "MD5 (zr1394_5_s5_R1.fastq.gz) = bec03cfacc459aa73798b8383a8cd2a9\tMD5 (Data/MBD/zr1394_5_s5_R1.fastq.gz) = bec03cfacc459aa73798b8383a8cd2a9\r\n",
      "MD5 (zr1394_5_s6_R1.fastq.gz) = 58162f7e654529bd6bb64846b5b41955\tMD5 (Data/MBD/zr1394_5_s6_R1.fastq.gz) = 58162f7e654529bd6bb64846b5b41955\r\n",
      "MD5 (zr1394_6_s1_R1.fastq.gz) = 32c2fff961f6a7ecaa17c364b274237f\tMD5 (Data/MBD/zr1394_6_s1_R1.fastq.gz) = 32c2fff961f6a7ecaa17c364b274237f\r\n",
      "MD5 (zr1394_6_s2_R1.fastq.gz) = 1eb423573b96f9545c143f3ca9b54a2a\tMD5 (Data/MBD/zr1394_6_s2_R1.fastq.gz) = 1eb423573b96f9545c143f3ca9b54a2a\r\n",
      "MD5 (zr1394_6_s3_R1.fastq.gz) = 34785d0abec0b6f5b07c02a05e49d371\tMD5 (Data/MBD/zr1394_6_s3_R1.fastq.gz) = 34785d0abec0b6f5b07c02a05e49d371\r\n",
      "MD5 (zr1394_6_s4_R1.fastq.gz) = 3c2950ef9f2a1d75fa327ca2d0352ab8\tMD5 (Data/MBD/zr1394_6_s4_R1.fastq.gz) = 3c2950ef9f2a1d75fa327ca2d0352ab8\r\n",
      "MD5 (zr1394_6_s5_R1.fastq.gz) = a4ff9322d7fb4dcd2bf41ce74d74b4e5\tMD5 (Data/MBD/zr1394_6_s5_R1.fastq.gz) = a4ff9322d7fb4dcd2bf41ce74d74b4e5\r\n",
      "MD5 (zr1394_6_s6_R1.fastq.gz) = 526dfad8294a330a444bf60e06bdc622\tMD5 (Data/MBD/zr1394_6_s6_R1.fastq.gz) = 526dfad8294a330a444bf60e06bdc622\r\n",
      "MD5 (zr1394_7_s1_R1.fastq.gz) = 2540597fd38a7f768d3c5576b0c51589\tMD5 (Data/MBD/zr1394_7_s1_R1.fastq.gz) = 2540597fd38a7f768d3c5576b0c51589\r\n",
      "MD5 (zr1394_7_s2_R1.fastq.gz) = 9aca4d83d2efd8611fc8c2b15f9258c0\tMD5 (Data/MBD/zr1394_7_s2_R1.fastq.gz) = 9aca4d83d2efd8611fc8c2b15f9258c0\r\n",
      "MD5 (zr1394_7_s3_R1.fastq.gz) = 123458c626a77ba4518f4bbe78125dad\tMD5 (Data/MBD/zr1394_7_s3_R1.fastq.gz) = 123458c626a77ba4518f4bbe78125dad\r\n",
      "MD5 (zr1394_7_s4_R1.fastq.gz) = 4e50b279d19d5fb76aa61e9a38dc5ec1\tMD5 (Data/MBD/zr1394_7_s4_R1.fastq.gz) = 4e50b279d19d5fb76aa61e9a38dc5ec1\r\n",
      "MD5 (zr1394_7_s5_R1.fastq.gz) = 5957bdc46aaae1731ee79dddc99c15c5\tMD5 (Data/MBD/zr1394_7_s5_R1.fastq.gz) = 5957bdc46aaae1731ee79dddc99c15c5\r\n",
      "MD5 (zr1394_7_s6_R1.fastq.gz) = e496cc8c4bb18278c77464d45a3d49c4\tMD5 (Data/MBD/zr1394_7_s6_R1.fastq.gz) = e496cc8c4bb18278c77464d45a3d49c4\r\n",
      "MD5 (zr1394_8_s1_R1.fastq.gz) = f796e1d710d19399ea646948e64f881d\tMD5 (Data/MBD/zr1394_8_s1_R1.fastq.gz) = f796e1d710d19399ea646948e64f881d\r\n",
      "MD5 (zr1394_8_s2_R1.fastq.gz) = b9b3dcf27ccde3a54148daf04bbddcfd\tMD5 (Data/MBD/zr1394_8_s2_R1.fastq.gz) = b9b3dcf27ccde3a54148daf04bbddcfd\r\n",
      "MD5 (zr1394_8_s3_R1.fastq.gz) = 3fa868436c83cfea119704307f3fe11c\tMD5 (Data/MBD/zr1394_8_s3_R1.fastq.gz) = 3fa868436c83cfea119704307f3fe11c\r\n",
      "MD5 (zr1394_8_s4_R1.fastq.gz) = e5550ee2856d509e31e66f672abcb641\tMD5 (Data/MBD/zr1394_8_s4_R1.fastq.gz) = e5550ee2856d509e31e66f672abcb641\r\n",
      "MD5 (zr1394_8_s5_R1.fastq.gz) = a1d3225bef6c126293e45298f578e6b2\tMD5 (Data/MBD/zr1394_8_s5_R1.fastq.gz) = a1d3225bef6c126293e45298f578e6b2\r\n",
      "MD5 (zr1394_8_s6_R1.fastq.gz) = 9fb727b5e221ce1cd4cfd75e46cf30cf\tMD5 (Data/MBD/zr1394_8_s6_R1.fastq.gz) = 9fb727b5e221ce1cd4cfd75e46cf30cf\r\n",
      "MD5 (zr1394_9_s1_R1.fastq.gz) = d3b51f8451199c3e39cdac8d015cf989\tMD5 (Data/MBD/zr1394_9_s1_R1.fastq.gz) = d3b51f8451199c3e39cdac8d015cf989\r\n",
      "MD5 (zr1394_9_s2_R1.fastq.gz) = 004d8eb6231e7c32e5d6de4c78c28af6\tMD5 (Data/MBD/zr1394_9_s2_R1.fastq.gz) = 004d8eb6231e7c32e5d6de4c78c28af6\r\n",
      "MD5 (zr1394_9_s3_R1.fastq.gz) = d052ee9b6c88f9cd5d38bb2ad73cc118\tMD5 (Data/MBD/zr1394_9_s3_R1.fastq.gz) = d052ee9b6c88f9cd5d38bb2ad73cc118\r\n",
      "MD5 (zr1394_9_s4_R1.fastq.gz) = ed793f38ee309ab252a5a2e6b3c7d3d7\tMD5 (Data/MBD/zr1394_9_s4_R1.fastq.gz) = ed793f38ee309ab252a5a2e6b3c7d3d7\r\n",
      "MD5 (zr1394_9_s5_R1.fastq.gz) = f6092513aa5d7811d4207228bd5336e7\tMD5 (Data/MBD/zr1394_9_s5_R1.fastq.gz) = f6092513aa5d7811d4207228bd5336e7\r\n",
      "MD5 (zr1394_9_s6_R1.fastq.gz) = 5b0f480871406bbb5341bfcd2d95cec9\tMD5 (Data/MBD/zr1394_9_s6_R1.fastq.gz) = 5b0f480871406bbb5341bfcd2d95cec9\r\n"
     ]
    }
   ],
   "source": [
    "!cat Data/MBD/checksums_comparison.md5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n",
      "same\r\n"
     ]
    }
   ],
   "source": [
    "!awk '{ if ($4 == $8) { print \"same\"; } else { print \"different\"; } }' < Data/MBD/checksums_comparison.md5 "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Run QC on the Raw MBD data"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Make a new directory for the QC of the raw data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: Data/Raw_MBD_QC: File exists\r\n"
     ]
    }
   ],
   "source": [
    "!mkdir Data/Raw_MBD_QC"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Run [FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) Version 0.11.5\n",
    "Andrews S. (2010). FastQC: a quality control tool for high throughput sequence data. Available online at: http://www.bioinformatics.babraham.ac.uk/projects/fastqc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of zr1394_10_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_10_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_10_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_10_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_10_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_10_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_10_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_10_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_10_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 100% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_10_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_10_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_10_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_11_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_11_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_11_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_11_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_11_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_11_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_11_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_11_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_11_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_11_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_11_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_11_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_12_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 100% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_12_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_12_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_12_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_12_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_12_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_12_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_12_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_12_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_12_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_12_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_12_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_13_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 100% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_13_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_13_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_13_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_13_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_13_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_13_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_13_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_13_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_13_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_13_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_13_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_14_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_14_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_14_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_14_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_14_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_14_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_14_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_14_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_14_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_14_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_14_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_14_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_15_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_15_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_15_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_15_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_15_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_15_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_15_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_15_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_15_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_15_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_15_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_15_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_16_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_16_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_16_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_16_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_16_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_16_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_16_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_16_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_16_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_16_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_16_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_16_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_17_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_17_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_17_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_17_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_17_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_17_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_17_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_17_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_17_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_17_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_17_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_17_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_18_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_18_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_18_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_18_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_18_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_18_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_18_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_18_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_18_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_18_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_18_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_18_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_1_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_1_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_1_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_1_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_1_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_1_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_1_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_1_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_1_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_1_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_1_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_1_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_2_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 100% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_2_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_2_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_2_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_2_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_2_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_2_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_2_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_2_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_2_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_2_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_2_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_3_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_3_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_3_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_3_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_3_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_3_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_3_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_3_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_3_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_3_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_3_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_3_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_4_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_4_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_4_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_4_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_4_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_4_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_4_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_4_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_4_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_4_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_4_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_4_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_5_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_5_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_5_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_5_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_5_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_5_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_5_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_5_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_5_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_5_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_5_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_5_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_6_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 100% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_6_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_6_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_6_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_6_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_6_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_6_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_6_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_6_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_6_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_6_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_6_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_7_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_7_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_7_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_7_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_7_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_7_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_7_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_7_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_7_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_7_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_7_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_7_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_8_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_8_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_8_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_8_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_8_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_8_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_8_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_8_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_8_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_8_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_8_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_8_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_9_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_9_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_9_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_9_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_9_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_9_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_9_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_9_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_9_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_9_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_9_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_9_s6_R1.fastq.gz\n"
     ]
    }
   ],
   "source": [
    "!/Users/hollie/Documents/BioInf_Programs/FastQC/fastqc Data/MBD/*R1.fastq.gz -o Data/Raw_MBD_QC"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Examine file quality by combining QC reports into one file with multiqc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[INFO   ]         multiqc : This is MultiQC v0.8\n",
      "[INFO   ]         multiqc : Template    : default\n",
      "[INFO   ]         multiqc : Searching 'Data/Raw_MBD_QC'\n",
      "[INFO   ]          fastqc : Found 108 reports\n",
      "[INFO   ]         multiqc : Report      : Data/Raw_MBD_QC/multiqc_report.html\n",
      "[INFO   ]         multiqc : Data        : Data/Raw_MBD_QC/multiqc_data\n",
      "[INFO   ]         multiqc : MultiQC complete\n"
     ]
    }
   ],
   "source": [
    "!multiqc Data/Raw_MBD_QC -o Data/Raw_MBD_QC"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Concatenate sets of 6 downloaded files into one per sample"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!mkdir Data/MBD/Joined_Data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_1_*.fastq.gz > Data/MBD/Joined_Data/zr1394_1_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_2_*.fastq.gz > Data/MBD/Joined_Data/zr1394_2_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_3_*.fastq.gz > Data/MBD/Joined_Data/zr1394_3_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_4_*.fastq.gz > Data/MBD/Joined_Data/zr1394_4_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_5_*.fastq.gz > Data/MBD/Joined_Data/zr1394_5_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_6_*.fastq.gz > Data/MBD/Joined_Data/zr1394_6_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_7_*.fastq.gz > Data/MBD/Joined_Data/zr1394_7_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_8_*.fastq.gz > Data/MBD/Joined_Data/zr1394_8_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_9_*.fastq.gz > Data/MBD/Joined_Data/zr1394_9_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_10_*.fastq.gz > Data/MBD/Joined_Data/zr1394_10_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_11_*.fastq.gz > Data/MBD/Joined_Data/zr1394_11_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_12_*.fastq.gz > Data/MBD/Joined_Data/zr1394_12_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_13_*.fastq.gz > Data/MBD/Joined_Data/zr1394_13_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_14_*.fastq.gz > Data/MBD/Joined_Data/zr1394_14_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_15_*.fastq.gz > Data/MBD/Joined_Data/zr1394_15_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_16_*.fastq.gz > Data/MBD/Joined_Data/zr1394_16_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_17_*.fastq.gz > Data/MBD/Joined_Data/zr1394_17_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/zr1394_18_*.fastq.gz > Data/MBD/Joined_Data/zr1394_18_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!zgrep -c \"@HW\" Data/MBD/Joined_Data/*.fastq.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Data/MBD/Joined_Data/zr1394_10_all.fastq.gz:113424382\n",
    "* Data/MBD/Joined_Data/zr1394_11_all.fastq.gz:77274077\n",
    "* Data/MBD/Joined_Data/zr1394_12_all.fastq.gz:87308544\n",
    "* Data/MBD/Joined_Data/zr1394_13_all.fastq.gz:92858760\n",
    "* Data/MBD/Joined_Data/zr1394_14_all.fastq.gz:126836761\n",
    "* Data/MBD/Joined_Data/zr1394_15_all.fastq.gz:99431044\n",
    "* Data/MBD/Joined_Data/zr1394_16_all.fastq.gz:107049378\n",
    "* Data/MBD/Joined_Data/zr1394_17_all.fastq.gz:60202000\n",
    "* Data/MBD/Joined_Data/zr1394_18_all.fastq.gz:78526645\n",
    "* Data/MBD/Joined_Data/zr1394_1_all.fastq.gz:67384169\n",
    "* Data/MBD/Joined_Data/zr1394_2_all.fastq.gz:66415224\n",
    "* Data/MBD/Joined_Data/zr1394_3_all.fastq.gz:69101776\n",
    "* Data/MBD/Joined_Data/zr1394_4_all.fastq.gz:64145277\n",
    "* Data/MBD/Joined_Data/zr1394_5_all.fastq.gz:68281804\n",
    "* Data/MBD/Joined_Data/zr1394_6_all.fastq.gz:65443407\n",
    "* Data/MBD/Joined_Data/zr1394_7_all.fastq.gz:71533039\n",
    "* Data/MBD/Joined_Data/zr1394_8_all.fastq.gz:56320414\n",
    "* Data/MBD/Joined_Data/zr1394_9_all.fastq.gz:110300174"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!zgrep -c \"@HW\" Data/MBD/*.fastq.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Data/MBD/zr1394_10_s1_R1.fastq.gz:12129726\n",
    "* Data/MBD/zr1394_10_s2_R1.fastq.gz:11482874\n",
    "* Data/MBD/zr1394_10_s3_R1.fastq.gz:10547199\n",
    "* Data/MBD/zr1394_10_s4_R1.fastq.gz:26306980\n",
    "* Data/MBD/zr1394_10_s5_R1.fastq.gz:26573036\n",
    "* Data/MBD/zr1394_10_s6_R1.fastq.gz:26384567\n",
    "* Data/MBD/zr1394_11_s1_R1.fastq.gz:7893266\n",
    "* Data/MBD/zr1394_11_s2_R1.fastq.gz:7495554\n",
    "* Data/MBD/zr1394_11_s3_R1.fastq.gz:6775729\n",
    "* Data/MBD/zr1394_11_s4_R1.fastq.gz:18477911\n",
    "* Data/MBD/zr1394_11_s5_R1.fastq.gz:18305906\n",
    "* Data/MBD/zr1394_11_s6_R1.fastq.gz:18325711\n",
    "* Data/MBD/zr1394_12_s1_R1.fastq.gz:9669005\n",
    "* Data/MBD/zr1394_12_s2_R1.fastq.gz:9147875\n",
    "* Data/MBD/zr1394_12_s3_R1.fastq.gz:8338418\n",
    "* Data/MBD/zr1394_12_s4_R1.fastq.gz:20188420\n",
    "* Data/MBD/zr1394_12_s5_R1.fastq.gz:20006134\n",
    "* Data/MBD/zr1394_12_s6_R1.fastq.gz:19958692\n",
    "* Data/MBD/zr1394_13_s1_R1.fastq.gz:9840016\n",
    "* Data/MBD/zr1394_13_s2_R1.fastq.gz:9358447\n",
    "* Data/MBD/zr1394_13_s3_R1.fastq.gz:8578615\n",
    "* Data/MBD/zr1394_13_s4_R1.fastq.gz:21598254\n",
    "* Data/MBD/zr1394_13_s5_R1.fastq.gz:21823361\n",
    "* Data/MBD/zr1394_13_s6_R1.fastq.gz:21660067\n",
    "* Data/MBD/zr1394_14_s1_R1.fastq.gz:13367447\n",
    "* Data/MBD/zr1394_14_s2_R1.fastq.gz:12782967\n",
    "* Data/MBD/zr1394_14_s3_R1.fastq.gz:11914441\n",
    "* Data/MBD/zr1394_14_s4_R1.fastq.gz:29782244\n",
    "* Data/MBD/zr1394_14_s5_R1.fastq.gz:29515814\n",
    "* Data/MBD/zr1394_14_s6_R1.fastq.gz:29473848\n",
    "* Data/MBD/zr1394_15_s1_R1.fastq.gz:10879464\n",
    "* Data/MBD/zr1394_15_s2_R1.fastq.gz:10267923\n",
    "* Data/MBD/zr1394_15_s3_R1.fastq.gz:9425665\n",
    "* Data/MBD/zr1394_15_s4_R1.fastq.gz:23084106\n",
    "* Data/MBD/zr1394_15_s5_R1.fastq.gz:22853637\n",
    "* Data/MBD/zr1394_15_s6_R1.fastq.gz:22920249\n",
    "* Data/MBD/zr1394_16_s1_R1.fastq.gz:11242167\n",
    "* Data/MBD/zr1394_16_s2_R1.fastq.gz:10639584\n",
    "* Data/MBD/zr1394_16_s3_R1.fastq.gz:9717312\n",
    "* Data/MBD/zr1394_16_s4_R1.fastq.gz:25050434\n",
    "* Data/MBD/zr1394_16_s5_R1.fastq.gz:25317275\n",
    "* Data/MBD/zr1394_16_s6_R1.fastq.gz:25082606\n",
    "* Data/MBD/zr1394_17_s1_R1.fastq.gz:6292243\n",
    "* Data/MBD/zr1394_17_s2_R1.fastq.gz:5878505\n",
    "* Data/MBD/zr1394_17_s3_R1.fastq.gz:5300767\n",
    "* Data/MBD/zr1394_17_s4_R1.fastq.gz:14329557\n",
    "* Data/MBD/zr1394_17_s5_R1.fastq.gz:14186344\n",
    "* Data/MBD/zr1394_17_s6_R1.fastq.gz:14214584\n",
    "* Data/MBD/zr1394_18_s1_R1.fastq.gz:8197198\n",
    "* Data/MBD/zr1394_18_s2_R1.fastq.gz:7871675\n",
    "* Data/MBD/zr1394_18_s3_R1.fastq.gz:6898091\n",
    "* Data/MBD/zr1394_18_s4_R1.fastq.gz:18627445\n",
    "* Data/MBD/zr1394_18_s5_R1.fastq.gz:18447543\n",
    "* Data/MBD/zr1394_18_s6_R1.fastq.gz:18484693\n",
    "* Data/MBD/zr1394_1_s1_R1.fastq.gz:6768171\n",
    "* Data/MBD/zr1394_1_s2_R1.fastq.gz:7088611\n",
    "* Data/MBD/zr1394_1_s3_R1.fastq.gz:6127974\n",
    "* Data/MBD/zr1394_1_s4_R1.fastq.gz:15875631\n",
    "* Data/MBD/zr1394_1_s5_R1.fastq.gz:15747941\n",
    "* Data/MBD/zr1394_1_s6_R1.fastq.gz:15775841\n",
    "* Data/MBD/zr1394_2_s1_R1.fastq.gz:7073025\n",
    "* Data/MBD/zr1394_2_s2_R1.fastq.gz:6596118\n",
    "* Data/MBD/zr1394_2_s3_R1.fastq.gz:6079643\n",
    "* Data/MBD/zr1394_2_s4_R1.fastq.gz:15635185\n",
    "* Data/MBD/zr1394_2_s5_R1.fastq.gz:15492802\n",
    "* Data/MBD/zr1394_2_s6_R1.fastq.gz:15538451\n",
    "* Data/MBD/zr1394_3_s1_R1.fastq.gz:7293076\n",
    "* Data/MBD/zr1394_3_s2_R1.fastq.gz:6737811\n",
    "* Data/MBD/zr1394_3_s3_R1.fastq.gz:6144269\n",
    "* Data/MBD/zr1394_3_s4_R1.fastq.gz:16379997\n",
    "* Data/MBD/zr1394_3_s5_R1.fastq.gz:16260168\n",
    "* Data/MBD/zr1394_3_s6_R1.fastq.gz:16286455\n",
    "* Data/MBD/zr1394_4_s1_R1.fastq.gz:6649632\n",
    "* Data/MBD/zr1394_4_s2_R1.fastq.gz:6270620\n",
    "* Data/MBD/zr1394_4_s3_R1.fastq.gz:5813100\n",
    "* Data/MBD/zr1394_4_s4_R1.fastq.gz:15241034\n",
    "* Data/MBD/zr1394_4_s5_R1.fastq.gz:15080117\n",
    "* Data/MBD/zr1394_4_s6_R1.fastq.gz:15090774\n",
    "* Data/MBD/zr1394_5_s1_R1.fastq.gz:7149560\n",
    "* Data/MBD/zr1394_5_s2_R1.fastq.gz:6855766\n",
    "* Data/MBD/zr1394_5_s3_R1.fastq.gz:6226422\n",
    "* Data/MBD/zr1394_5_s4_R1.fastq.gz:16122713\n",
    "* Data/MBD/zr1394_5_s5_R1.fastq.gz:15952921\n",
    "* Data/MBD/zr1394_5_s6_R1.fastq.gz:15974422\n",
    "* Data/MBD/zr1394_6_s1_R1.fastq.gz:6614004\n",
    "* Data/MBD/zr1394_6_s2_R1.fastq.gz:6355162\n",
    "* Data/MBD/zr1394_6_s3_R1.fastq.gz:5529830\n",
    "* Data/MBD/zr1394_6_s4_R1.fastq.gz:15735916\n",
    "* Data/MBD/zr1394_6_s5_R1.fastq.gz:15596312\n",
    "* Data/MBD/zr1394_6_s6_R1.fastq.gz:15612183\n",
    "* Data/MBD/zr1394_7_s1_R1.fastq.gz:7571742\n",
    "* Data/MBD/zr1394_7_s2_R1.fastq.gz:7125072\n",
    "* Data/MBD/zr1394_7_s3_R1.fastq.gz:6436872\n",
    "* Data/MBD/zr1394_7_s4_R1.fastq.gz:16920844\n",
    "* Data/MBD/zr1394_7_s5_R1.fastq.gz:16731051\n",
    "* Data/MBD/zr1394_7_s6_R1.fastq.gz:16747458\n",
    "* Data/MBD/zr1394_8_s1_R1.fastq.gz:6146166\n",
    "* Data/MBD/zr1394_8_s2_R1.fastq.gz:5775399\n",
    "* Data/MBD/zr1394_8_s3_R1.fastq.gz:5262335\n",
    "* Data/MBD/zr1394_8_s4_R1.fastq.gz:13117656\n",
    "* Data/MBD/zr1394_8_s5_R1.fastq.gz:12994067\n",
    "* Data/MBD/zr1394_8_s6_R1.fastq.gz:13024791\n",
    "* Data/MBD/zr1394_9_s1_R1.fastq.gz:11259296\n",
    "* Data/MBD/zr1394_9_s2_R1.fastq.gz:11063708\n",
    "* Data/MBD/zr1394_9_s3_R1.fastq.gz:9830662\n",
    "* Data/MBD/zr1394_9_s4_R1.fastq.gz:26183621\n",
    "* Data/MBD/zr1394_9_s5_R1.fastq.gz:25958649\n",
    "* Data/MBD/zr1394_9_s6_R1.fastq.gz:26004238"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!gunzip Data/MBD/Joined_Data/*.fastq.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Trim adapters and low quality sequences to generate cleaned data files"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Make a new directory for the cleaned data files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: Data/MBD/Clean_Data: File exists\r\n"
     ]
    }
   ],
   "source": [
    "!mkdir Data/MBD/Clean_Data"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Make file of Truseq adapters"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "echo \\\n",
    "'>TruSeq3_IndexedAdapter\n",
    "AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC\n",
    ">TruSeq3_UniversalAdapter\n",
    "AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA' > Data/TruSeq3-SE.fa"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">TruSeq3_IndexedAdapter\r\n",
      "AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC\r\n",
      ">TruSeq3_UniversalAdapter\r\n",
      "AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA\r\n"
     ]
    }
   ],
   "source": [
    "!head Data/TruSeq3-SE.fa"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Loop files to complete trimmomatic on all sample read files\n",
    "* Remove Illumina adapters: ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10\n",
    "* Remove leading low quality or N bases below quality =3: LEADING:3\n",
    "* Remove trailing low quality or N bases below quality =3: TRAILING:3\n",
    "* Scan with a sliding window of 4 bases, cutting when  average quality per base is below 15: SLIDINGWINDOW:4:15\n",
    "* Remove reads below 20bp long: MINLEN:20;\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_10_s1_R1.fastq.gz cleaned_zr1394_10_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 12129726 Surviving: 10441477 (86.08%) Dropped: 1688249 (13.92%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_10_s2_R1.fastq.gz cleaned_zr1394_10_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 11482874 Surviving: 10123890 (88.17%) Dropped: 1358984 (11.83%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_10_s3_R1.fastq.gz cleaned_zr1394_10_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 10547199 Surviving: 9474454 (89.83%) Dropped: 1072745 (10.17%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_10_s4_R1.fastq.gz cleaned_zr1394_10_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 26306980 Surviving: 24760860 (94.12%) Dropped: 1546120 (5.88%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_10_s5_R1.fastq.gz cleaned_zr1394_10_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 26573036 Surviving: 25046673 (94.26%) Dropped: 1526363 (5.74%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_10_s6_R1.fastq.gz cleaned_zr1394_10_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 26384567 Surviving: 24847543 (94.17%) Dropped: 1537024 (5.83%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_11_s1_R1.fastq.gz cleaned_zr1394_11_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7893266 Surviving: 6852625 (86.82%) Dropped: 1040641 (13.18%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_11_s2_R1.fastq.gz cleaned_zr1394_11_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7495554 Surviving: 6662076 (88.88%) Dropped: 833478 (11.12%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_11_s3_R1.fastq.gz cleaned_zr1394_11_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6775729 Surviving: 6125516 (90.40%) Dropped: 650213 (9.60%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_11_s4_R1.fastq.gz cleaned_zr1394_11_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 18477911 Surviving: 17475776 (94.58%) Dropped: 1002135 (5.42%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_11_s5_R1.fastq.gz cleaned_zr1394_11_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 18305906 Surviving: 17291115 (94.46%) Dropped: 1014791 (5.54%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_11_s6_R1.fastq.gz cleaned_zr1394_11_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 18325711 Surviving: 17321808 (94.52%) Dropped: 1003903 (5.48%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_12_s1_R1.fastq.gz cleaned_zr1394_12_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 9669005 Surviving: 8343594 (86.29%) Dropped: 1325411 (13.71%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_12_s2_R1.fastq.gz cleaned_zr1394_12_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 9147875 Surviving: 8067647 (88.19%) Dropped: 1080228 (11.81%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_12_s3_R1.fastq.gz cleaned_zr1394_12_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 8338418 Surviving: 7489976 (89.82%) Dropped: 848442 (10.18%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_12_s4_R1.fastq.gz cleaned_zr1394_12_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 20188420 Surviving: 19028902 (94.26%) Dropped: 1159518 (5.74%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_12_s5_R1.fastq.gz cleaned_zr1394_12_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 20006134 Surviving: 18835156 (94.15%) Dropped: 1170978 (5.85%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_12_s6_R1.fastq.gz cleaned_zr1394_12_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 19958692 Surviving: 18777654 (94.08%) Dropped: 1181038 (5.92%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_13_s1_R1.fastq.gz cleaned_zr1394_13_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 9840016 Surviving: 8607193 (87.47%) Dropped: 1232823 (12.53%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_13_s2_R1.fastq.gz cleaned_zr1394_13_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 9358447 Surviving: 8364117 (89.38%) Dropped: 994330 (10.62%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_13_s3_R1.fastq.gz cleaned_zr1394_13_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 8578615 Surviving: 7803622 (90.97%) Dropped: 774993 (9.03%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_13_s4_R1.fastq.gz cleaned_zr1394_13_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 21598254 Surviving: 20439482 (94.63%) Dropped: 1158772 (5.37%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_13_s5_R1.fastq.gz cleaned_zr1394_13_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 21823361 Surviving: 20682194 (94.77%) Dropped: 1141167 (5.23%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_13_s6_R1.fastq.gz cleaned_zr1394_13_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 21660067 Surviving: 20510728 (94.69%) Dropped: 1149339 (5.31%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_14_s1_R1.fastq.gz cleaned_zr1394_14_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 13367447 Surviving: 11516118 (86.15%) Dropped: 1851329 (13.85%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_14_s2_R1.fastq.gz cleaned_zr1394_14_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 12782967 Surviving: 11262884 (88.11%) Dropped: 1520083 (11.89%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_14_s3_R1.fastq.gz cleaned_zr1394_14_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 11914441 Surviving: 10707861 (89.87%) Dropped: 1206580 (10.13%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_14_s4_R1.fastq.gz cleaned_zr1394_14_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 29782244 Surviving: 28102235 (94.36%) Dropped: 1680009 (5.64%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_14_s5_R1.fastq.gz cleaned_zr1394_14_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 29515814 Surviving: 27825679 (94.27%) Dropped: 1690135 (5.73%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_14_s6_R1.fastq.gz cleaned_zr1394_14_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 29473848 Surviving: 27768518 (94.21%) Dropped: 1705330 (5.79%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_15_s1_R1.fastq.gz cleaned_zr1394_15_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 10879464 Surviving: 9382482 (86.24%) Dropped: 1496982 (13.76%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_15_s2_R1.fastq.gz cleaned_zr1394_15_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 10267923 Surviving: 9052599 (88.16%) Dropped: 1215324 (11.84%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_15_s3_R1.fastq.gz cleaned_zr1394_15_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 9425665 Surviving: 8455241 (89.70%) Dropped: 970424 (10.30%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_15_s4_R1.fastq.gz cleaned_zr1394_15_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 23084106 Surviving: 21752869 (94.23%) Dropped: 1331237 (5.77%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_15_s5_R1.fastq.gz cleaned_zr1394_15_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 22853637 Surviving: 21502281 (94.09%) Dropped: 1351356 (5.91%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_15_s6_R1.fastq.gz cleaned_zr1394_15_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 22920249 Surviving: 21580427 (94.15%) Dropped: 1339822 (5.85%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_16_s1_R1.fastq.gz cleaned_zr1394_16_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 11242167 Surviving: 9754481 (86.77%) Dropped: 1487686 (13.23%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_16_s2_R1.fastq.gz cleaned_zr1394_16_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 10639584 Surviving: 9438647 (88.71%) Dropped: 1200937 (11.29%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_16_s3_R1.fastq.gz cleaned_zr1394_16_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 9717312 Surviving: 8779415 (90.35%) Dropped: 937897 (9.65%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_16_s4_R1.fastq.gz cleaned_zr1394_16_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 25050434 Surviving: 23615664 (94.27%) Dropped: 1434770 (5.73%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_16_s5_R1.fastq.gz cleaned_zr1394_16_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 25317275 Surviving: 23906002 (94.43%) Dropped: 1411273 (5.57%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_16_s6_R1.fastq.gz cleaned_zr1394_16_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 25082606 Surviving: 23662346 (94.34%) Dropped: 1420260 (5.66%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_17_s1_R1.fastq.gz cleaned_zr1394_17_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6292243 Surviving: 5452289 (86.65%) Dropped: 839954 (13.35%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_17_s2_R1.fastq.gz cleaned_zr1394_17_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 5878505 Surviving: 5224892 (88.88%) Dropped: 653613 (11.12%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_17_s3_R1.fastq.gz cleaned_zr1394_17_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 5300767 Surviving: 4792356 (90.41%) Dropped: 508411 (9.59%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_17_s4_R1.fastq.gz cleaned_zr1394_17_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 14329557 Surviving: 13537941 (94.48%) Dropped: 791616 (5.52%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_17_s5_R1.fastq.gz cleaned_zr1394_17_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 14186344 Surviving: 13388541 (94.38%) Dropped: 797803 (5.62%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_17_s6_R1.fastq.gz cleaned_zr1394_17_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 14214584 Surviving: 13424568 (94.44%) Dropped: 790016 (5.56%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_18_s1_R1.fastq.gz cleaned_zr1394_18_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 8197198 Surviving: 7205960 (87.91%) Dropped: 991238 (12.09%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_18_s2_R1.fastq.gz cleaned_zr1394_18_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7871675 Surviving: 7067776 (89.79%) Dropped: 803899 (10.21%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_18_s3_R1.fastq.gz cleaned_zr1394_18_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6898091 Surviving: 6298989 (91.31%) Dropped: 599102 (8.69%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_18_s4_R1.fastq.gz cleaned_zr1394_18_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 18627445 Surviving: 17702415 (95.03%) Dropped: 925030 (4.97%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_18_s5_R1.fastq.gz cleaned_zr1394_18_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 18447543 Surviving: 17512007 (94.93%) Dropped: 935536 (5.07%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_18_s6_R1.fastq.gz cleaned_zr1394_18_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 18484693 Surviving: 17562879 (95.01%) Dropped: 921814 (4.99%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_1_s1_R1.fastq.gz cleaned_zr1394_1_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6768171 Surviving: 6074160 (89.75%) Dropped: 694011 (10.25%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_1_s2_R1.fastq.gz cleaned_zr1394_1_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7088611 Surviving: 6223799 (87.80%) Dropped: 864812 (12.20%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_1_s3_R1.fastq.gz cleaned_zr1394_1_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6127974 Surviving: 5590584 (91.23%) Dropped: 537390 (8.77%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_1_s4_R1.fastq.gz cleaned_zr1394_1_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15875631 Surviving: 15076683 (94.97%) Dropped: 798948 (5.03%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_1_s5_R1.fastq.gz cleaned_zr1394_1_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15747941 Surviving: 14932739 (94.82%) Dropped: 815202 (5.18%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_1_s6_R1.fastq.gz cleaned_zr1394_1_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15775841 Surviving: 14970282 (94.89%) Dropped: 805559 (5.11%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_2_s1_R1.fastq.gz cleaned_zr1394_2_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7073025 Surviving: 6171777 (87.26%) Dropped: 901248 (12.74%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_2_s2_R1.fastq.gz cleaned_zr1394_2_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6596118 Surviving: 5890562 (89.30%) Dropped: 705556 (10.70%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_2_s3_R1.fastq.gz cleaned_zr1394_2_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6079643 Surviving: 5521250 (90.82%) Dropped: 558393 (9.18%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_2_s4_R1.fastq.gz cleaned_zr1394_2_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15635185 Surviving: 14801515 (94.67%) Dropped: 833670 (5.33%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_2_s5_R1.fastq.gz cleaned_zr1394_2_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15492802 Surviving: 14649711 (94.56%) Dropped: 843091 (5.44%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_2_s6_R1.fastq.gz cleaned_zr1394_2_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15538451 Surviving: 14703208 (94.62%) Dropped: 835243 (5.38%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_3_s1_R1.fastq.gz cleaned_zr1394_3_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7293076 Surviving: 6367353 (87.31%) Dropped: 925723 (12.69%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_3_s2_R1.fastq.gz cleaned_zr1394_3_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6737811 Surviving: 6031667 (89.52%) Dropped: 706144 (10.48%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_3_s3_R1.fastq.gz cleaned_zr1394_3_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6144269 Surviving: 5584683 (90.89%) Dropped: 559586 (9.11%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_3_s4_R1.fastq.gz cleaned_zr1394_3_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 16379997 Surviving: 15508074 (94.68%) Dropped: 871923 (5.32%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_3_s5_R1.fastq.gz cleaned_zr1394_3_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 16260168 Surviving: 15378237 (94.58%) Dropped: 881931 (5.42%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_3_s6_R1.fastq.gz cleaned_zr1394_3_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 16286455 Surviving: 15415276 (94.65%) Dropped: 871179 (5.35%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_4_s1_R1.fastq.gz cleaned_zr1394_4_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6649632 Surviving: 5835927 (87.76%) Dropped: 813705 (12.24%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_4_s2_R1.fastq.gz cleaned_zr1394_4_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6270620 Surviving: 5619396 (89.61%) Dropped: 651224 (10.39%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_4_s3_R1.fastq.gz cleaned_zr1394_4_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 5813100 Surviving: 5293583 (91.06%) Dropped: 519517 (8.94%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_4_s4_R1.fastq.gz cleaned_zr1394_4_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15241034 Surviving: 14449163 (94.80%) Dropped: 791871 (5.20%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_4_s5_R1.fastq.gz cleaned_zr1394_4_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15080117 Surviving: 14271465 (94.64%) Dropped: 808652 (5.36%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_4_s6_R1.fastq.gz cleaned_zr1394_4_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15090774 Surviving: 14289906 (94.69%) Dropped: 800868 (5.31%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_5_s1_R1.fastq.gz cleaned_zr1394_5_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7149560 Surviving: 6283056 (87.88%) Dropped: 866504 (12.12%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_5_s2_R1.fastq.gz cleaned_zr1394_5_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6855766 Surviving: 6158545 (89.83%) Dropped: 697221 (10.17%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_5_s3_R1.fastq.gz cleaned_zr1394_5_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6226422 Surviving: 5691560 (91.41%) Dropped: 534862 (8.59%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_5_s4_R1.fastq.gz cleaned_zr1394_5_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 16122713 Surviving: 15307920 (94.95%) Dropped: 814793 (5.05%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_5_s5_R1.fastq.gz cleaned_zr1394_5_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15952921 Surviving: 15121260 (94.79%) Dropped: 831661 (5.21%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_5_s6_R1.fastq.gz cleaned_zr1394_5_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15974422 Surviving: 15154485 (94.87%) Dropped: 819937 (5.13%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_6_s1_R1.fastq.gz cleaned_zr1394_6_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6614004 Surviving: 5824794 (88.07%) Dropped: 789210 (11.93%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_6_s2_R1.fastq.gz cleaned_zr1394_6_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6355162 Surviving: 5725069 (90.09%) Dropped: 630093 (9.91%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_6_s3_R1.fastq.gz cleaned_zr1394_6_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 5529830 Surviving: 5060678 (91.52%) Dropped: 469152 (8.48%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_6_s4_R1.fastq.gz cleaned_zr1394_6_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15735916 Surviving: 14962344 (95.08%) Dropped: 773572 (4.92%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_6_s5_R1.fastq.gz cleaned_zr1394_6_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15596312 Surviving: 14807873 (94.94%) Dropped: 788439 (5.06%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_6_s6_R1.fastq.gz cleaned_zr1394_6_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15612183 Surviving: 14830851 (95.00%) Dropped: 781332 (5.00%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_7_s1_R1.fastq.gz cleaned_zr1394_7_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7571742 Surviving: 6636263 (87.65%) Dropped: 935479 (12.35%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_7_s2_R1.fastq.gz cleaned_zr1394_7_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7125072 Surviving: 6381643 (89.57%) Dropped: 743429 (10.43%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_7_s3_R1.fastq.gz cleaned_zr1394_7_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6436872 Surviving: 5863521 (91.09%) Dropped: 573351 (8.91%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_7_s4_R1.fastq.gz cleaned_zr1394_7_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 16920844 Surviving: 16043877 (94.82%) Dropped: 876967 (5.18%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_7_s5_R1.fastq.gz cleaned_zr1394_7_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 16731051 Surviving: 15838006 (94.66%) Dropped: 893045 (5.34%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_7_s6_R1.fastq.gz cleaned_zr1394_7_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 16747458 Surviving: 15863522 (94.72%) Dropped: 883936 (5.28%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_8_s1_R1.fastq.gz cleaned_zr1394_8_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6146166 Surviving: 5354370 (87.12%) Dropped: 791796 (12.88%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_8_s2_R1.fastq.gz cleaned_zr1394_8_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 5775399 Surviving: 5151441 (89.20%) Dropped: 623958 (10.80%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_8_s3_R1.fastq.gz cleaned_zr1394_8_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 5262335 Surviving: 4780667 (90.85%) Dropped: 481668 (9.15%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_8_s4_R1.fastq.gz cleaned_zr1394_8_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 13117656 Surviving: 12424238 (94.71%) Dropped: 693418 (5.29%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_8_s5_R1.fastq.gz cleaned_zr1394_8_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 12994067 Surviving: 12290612 (94.59%) Dropped: 703455 (5.41%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_8_s6_R1.fastq.gz cleaned_zr1394_8_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 13024791 Surviving: 12327811 (94.65%) Dropped: 696980 (5.35%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_9_s1_R1.fastq.gz cleaned_zr1394_9_s1_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 11259296 Surviving: 9885315 (87.80%) Dropped: 1373981 (12.20%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_9_s2_R1.fastq.gz cleaned_zr1394_9_s2_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 11063708 Surviving: 9926805 (89.72%) Dropped: 1136903 (10.28%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_9_s3_R1.fastq.gz cleaned_zr1394_9_s3_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 9830662 Surviving: 8976011 (91.31%) Dropped: 854651 (8.69%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_9_s4_R1.fastq.gz cleaned_zr1394_9_s4_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 26183621 Surviving: 24894836 (95.08%) Dropped: 1288785 (4.92%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_9_s5_R1.fastq.gz cleaned_zr1394_9_s5_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 25958649 Surviving: 24644249 (94.94%) Dropped: 1314400 (5.06%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 Data/MBD/zr1394_9_s6_R1.fastq.gz cleaned_zr1394_9_s6_R1.fastq.gz ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 26004238 Surviving: 24705402 (95.01%) Dropped: 1298836 (4.99%)\n",
      "TrimmomaticSE: Completed successfully\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for file in Data/MBD/*R1.fastq.gz\n",
    "do\n",
    "newname=${file##*/} \n",
    "java -jar /Users/hollie/Documents/BioInf_Programs/Trimmomatic-0.36/trimmomatic-0.36.jar \\\n",
    "SE \\\n",
    "-threads 16 \\\n",
    "-phred33 \"$file\" \\\n",
    "cleaned_$newname \\\n",
    "ILLUMINACLIP:Data/TruSeq3-SE.fa:2:30:10 \\\n",
    "LEADING:3 \\\n",
    "TRAILING:3 \\\n",
    "SLIDINGWINDOW:4:15 \\\n",
    "MINLEN:20;\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Move trimmed data to clean data folder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!mv *.fastq.gz Data/MBD/Clean_Data/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Run QC on the cleaned MBD files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: Data/Cleaned_MBD_QC: File exists\r\n"
     ]
    }
   ],
   "source": [
    "!mkdir Data/Cleaned_MBD_QC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {
    "collapsed": false,
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 100% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 100% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 100% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_9_s6_R1.fastq.gz\n"
     ]
    }
   ],
   "source": [
    "!/Users/hollie/Documents/BioInf_Programs/FastQC/fastqc Data/MBD/Clean_Data/*R1.fastq.gz -o Data/Cleaned_MBD_QC"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Examine file quality by combining QC reports into one file with multiqc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[INFO   ]         multiqc : This is MultiQC v0.8\n",
      "[INFO   ]         multiqc : Template    : default\n",
      "[INFO   ]         multiqc : Searching 'Data/Cleaned_MBD_QC/'\n",
      "[INFO   ]          fastqc : Found 108 reports\n",
      "[INFO   ]         multiqc : Report      : Data/Cleaned_MBD_QC/multiqc_report.html\n",
      "[INFO   ]         multiqc : Data        : Data/Cleaned_MBD_QC/multiqc_data\n",
      "[INFO   ]         multiqc : MultiQC complete\n"
     ]
    }
   ],
   "source": [
    "!multiqc Data/Cleaned_MBD_QC/ -o Data/Cleaned_MBD_QC"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Count Number of sequences per sample in raw data files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Data/MBD/zr1394_10_s1_R1.fastq.gz:12129726\n",
      "Data/MBD/zr1394_10_s2_R1.fastq.gz:11482874\n",
      "Data/MBD/zr1394_10_s3_R1.fastq.gz:10547199\n",
      "Data/MBD/zr1394_10_s4_R1.fastq.gz:26306980\n",
      "Data/MBD/zr1394_10_s5_R1.fastq.gz:26573036\n",
      "Data/MBD/zr1394_10_s6_R1.fastq.gz:26384567\n",
      "Data/MBD/zr1394_11_s1_R1.fastq.gz:7893266\n",
      "Data/MBD/zr1394_11_s2_R1.fastq.gz:7495554\n",
      "Data/MBD/zr1394_11_s3_R1.fastq.gz:6775729\n",
      "Data/MBD/zr1394_11_s4_R1.fastq.gz:18477911\n",
      "Data/MBD/zr1394_11_s5_R1.fastq.gz:18305906\n",
      "Data/MBD/zr1394_11_s6_R1.fastq.gz:18325711\n",
      "Data/MBD/zr1394_12_s1_R1.fastq.gz:9669005\n",
      "Data/MBD/zr1394_12_s2_R1.fastq.gz:9147875\n",
      "Data/MBD/zr1394_12_s3_R1.fastq.gz:8338418\n",
      "Data/MBD/zr1394_12_s4_R1.fastq.gz:20188420\n",
      "Data/MBD/zr1394_12_s5_R1.fastq.gz:20006134\n",
      "Data/MBD/zr1394_12_s6_R1.fastq.gz:19958692\n",
      "Data/MBD/zr1394_13_s1_R1.fastq.gz:9840016\n",
      "Data/MBD/zr1394_13_s2_R1.fastq.gz:9358447\n",
      "Data/MBD/zr1394_13_s3_R1.fastq.gz:8578615\n",
      "Data/MBD/zr1394_13_s4_R1.fastq.gz:21598254\n",
      "Data/MBD/zr1394_13_s5_R1.fastq.gz:21823361\n",
      "Data/MBD/zr1394_13_s6_R1.fastq.gz:21660067\n",
      "Data/MBD/zr1394_14_s1_R1.fastq.gz:13367447\n",
      "Data/MBD/zr1394_14_s2_R1.fastq.gz:12782967\n",
      "Data/MBD/zr1394_14_s3_R1.fastq.gz:11914441\n",
      "Data/MBD/zr1394_14_s4_R1.fastq.gz:29782244\n",
      "Data/MBD/zr1394_14_s5_R1.fastq.gz:29515814\n",
      "Data/MBD/zr1394_14_s6_R1.fastq.gz:29473848\n",
      "Data/MBD/zr1394_15_s1_R1.fastq.gz:10879464\n",
      "Data/MBD/zr1394_15_s2_R1.fastq.gz:10267923\n",
      "Data/MBD/zr1394_15_s3_R1.fastq.gz:9425665\n",
      "Data/MBD/zr1394_15_s4_R1.fastq.gz:23084106\n",
      "Data/MBD/zr1394_15_s5_R1.fastq.gz:22853637\n",
      "Data/MBD/zr1394_15_s6_R1.fastq.gz:22920249\n",
      "Data/MBD/zr1394_16_s1_R1.fastq.gz:11242167\n",
      "Data/MBD/zr1394_16_s2_R1.fastq.gz:10639584\n",
      "Data/MBD/zr1394_16_s3_R1.fastq.gz:9717312\n",
      "Data/MBD/zr1394_16_s4_R1.fastq.gz:25050434\n",
      "Data/MBD/zr1394_16_s5_R1.fastq.gz:25317275\n",
      "Data/MBD/zr1394_16_s6_R1.fastq.gz:25082606\n",
      "Data/MBD/zr1394_17_s1_R1.fastq.gz:6292243\n",
      "Data/MBD/zr1394_17_s2_R1.fastq.gz:5878505\n",
      "Data/MBD/zr1394_17_s3_R1.fastq.gz:5300767\n",
      "Data/MBD/zr1394_17_s4_R1.fastq.gz:14329557\n",
      "Data/MBD/zr1394_17_s5_R1.fastq.gz:14186344\n",
      "Data/MBD/zr1394_17_s6_R1.fastq.gz:14214584\n",
      "Data/MBD/zr1394_18_s1_R1.fastq.gz:8197198\n",
      "Data/MBD/zr1394_18_s2_R1.fastq.gz:7871675\n",
      "Data/MBD/zr1394_18_s3_R1.fastq.gz:6898091\n",
      "Data/MBD/zr1394_18_s4_R1.fastq.gz:18627445\n",
      "Data/MBD/zr1394_18_s5_R1.fastq.gz:18447543\n",
      "Data/MBD/zr1394_18_s6_R1.fastq.gz:18484693\n",
      "Data/MBD/zr1394_1_s1_R1.fastq.gz:6768171\n",
      "Data/MBD/zr1394_1_s2_R1.fastq.gz:7088611\n",
      "Data/MBD/zr1394_1_s3_R1.fastq.gz:6127974\n",
      "Data/MBD/zr1394_1_s4_R1.fastq.gz:15875631\n",
      "Data/MBD/zr1394_1_s5_R1.fastq.gz:15747941\n",
      "Data/MBD/zr1394_1_s6_R1.fastq.gz:15775841\n",
      "Data/MBD/zr1394_2_s1_R1.fastq.gz:7073025\n",
      "Data/MBD/zr1394_2_s2_R1.fastq.gz:6596118\n",
      "Data/MBD/zr1394_2_s3_R1.fastq.gz:6079643\n",
      "Data/MBD/zr1394_2_s4_R1.fastq.gz:15635185\n",
      "Data/MBD/zr1394_2_s5_R1.fastq.gz:15492802\n",
      "Data/MBD/zr1394_2_s6_R1.fastq.gz:15538451\n",
      "Data/MBD/zr1394_3_s1_R1.fastq.gz:7293076\n",
      "Data/MBD/zr1394_3_s2_R1.fastq.gz:6737811\n",
      "Data/MBD/zr1394_3_s3_R1.fastq.gz:6144269\n",
      "Data/MBD/zr1394_3_s4_R1.fastq.gz:16379997\n",
      "Data/MBD/zr1394_3_s5_R1.fastq.gz:16260168\n",
      "Data/MBD/zr1394_3_s6_R1.fastq.gz:16286455\n",
      "Data/MBD/zr1394_4_s1_R1.fastq.gz:6649632\n",
      "Data/MBD/zr1394_4_s2_R1.fastq.gz:6270620\n",
      "Data/MBD/zr1394_4_s3_R1.fastq.gz:5813100\n",
      "Data/MBD/zr1394_4_s4_R1.fastq.gz:15241034\n",
      "Data/MBD/zr1394_4_s5_R1.fastq.gz:15080117\n",
      "Data/MBD/zr1394_4_s6_R1.fastq.gz:15090774\n",
      "Data/MBD/zr1394_5_s1_R1.fastq.gz:7149560\n",
      "Data/MBD/zr1394_5_s2_R1.fastq.gz:6855766\n",
      "Data/MBD/zr1394_5_s3_R1.fastq.gz:6226422\n",
      "Data/MBD/zr1394_5_s4_R1.fastq.gz:16122713\n",
      "Data/MBD/zr1394_5_s5_R1.fastq.gz:15952921\n",
      "Data/MBD/zr1394_5_s6_R1.fastq.gz:15974422\n",
      "Data/MBD/zr1394_6_s1_R1.fastq.gz:6614004\n",
      "Data/MBD/zr1394_6_s2_R1.fastq.gz:6355162\n",
      "Data/MBD/zr1394_6_s3_R1.fastq.gz:5529830\n",
      "Data/MBD/zr1394_6_s4_R1.fastq.gz:15735916\n",
      "Data/MBD/zr1394_6_s5_R1.fastq.gz:15596312\n",
      "Data/MBD/zr1394_6_s6_R1.fastq.gz:15612183\n",
      "Data/MBD/zr1394_7_s1_R1.fastq.gz:7571742\n",
      "Data/MBD/zr1394_7_s2_R1.fastq.gz:7125072\n",
      "Data/MBD/zr1394_7_s3_R1.fastq.gz:6436872\n",
      "Data/MBD/zr1394_7_s4_R1.fastq.gz:16920844\n",
      "Data/MBD/zr1394_7_s5_R1.fastq.gz:16731051\n",
      "Data/MBD/zr1394_7_s6_R1.fastq.gz:16747458\n",
      "Data/MBD/zr1394_8_s1_R1.fastq.gz:6146166\n",
      "Data/MBD/zr1394_8_s2_R1.fastq.gz:5775399\n",
      "Data/MBD/zr1394_8_s3_R1.fastq.gz:5262335\n",
      "Data/MBD/zr1394_8_s4_R1.fastq.gz:13117656\n",
      "Data/MBD/zr1394_8_s5_R1.fastq.gz:12994067\n",
      "Data/MBD/zr1394_8_s6_R1.fastq.gz:13024791\n",
      "Data/MBD/zr1394_9_s1_R1.fastq.gz:11259296\n",
      "Data/MBD/zr1394_9_s2_R1.fastq.gz:11063708\n",
      "Data/MBD/zr1394_9_s3_R1.fastq.gz:9830662\n",
      "Data/MBD/zr1394_9_s4_R1.fastq.gz:26183621\n",
      "Data/MBD/zr1394_9_s5_R1.fastq.gz:25958649\n",
      "Data/MBD/zr1394_9_s6_R1.fastq.gz:26004238\n"
     ]
    }
   ],
   "source": [
    "!zgrep -c \"@HW\" Data/MBD/*.fastq.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Count Number of sequences per sample in cleaned data files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Data/MBD/Clean_Data/cleaned_zr1394_10_s1_R1.fastq.gz:10441477\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_10_s2_R1.fastq.gz:10123890\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_10_s3_R1.fastq.gz:9474454\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_10_s4_R1.fastq.gz:24760860\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_10_s5_R1.fastq.gz:25046673\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_10_s6_R1.fastq.gz:24847543\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_11_s1_R1.fastq.gz:6852625\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_11_s2_R1.fastq.gz:6662076\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_11_s3_R1.fastq.gz:6125516\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_11_s4_R1.fastq.gz:17475776\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_11_s5_R1.fastq.gz:17291115\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_11_s6_R1.fastq.gz:17321808\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_12_s1_R1.fastq.gz:8343594\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_12_s2_R1.fastq.gz:8067647\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_12_s3_R1.fastq.gz:7489976\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_12_s4_R1.fastq.gz:19028902\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_12_s5_R1.fastq.gz:18835156\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_12_s6_R1.fastq.gz:18777654\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_13_s1_R1.fastq.gz:8607193\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_13_s2_R1.fastq.gz:8364117\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_13_s3_R1.fastq.gz:7803622\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_13_s4_R1.fastq.gz:20439482\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_13_s5_R1.fastq.gz:20682194\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_13_s6_R1.fastq.gz:20510728\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_14_s1_R1.fastq.gz:11516118\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_14_s2_R1.fastq.gz:11262884\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_14_s3_R1.fastq.gz:10707861\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_14_s4_R1.fastq.gz:28102235\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_14_s5_R1.fastq.gz:27825679\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_14_s6_R1.fastq.gz:27768518\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_15_s1_R1.fastq.gz:9382482\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_15_s2_R1.fastq.gz:9052599\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_15_s3_R1.fastq.gz:8455241\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_15_s4_R1.fastq.gz:21752869\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_15_s5_R1.fastq.gz:21502281\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_15_s6_R1.fastq.gz:21580427\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_16_s1_R1.fastq.gz:9754481\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_16_s2_R1.fastq.gz:9438647\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_16_s3_R1.fastq.gz:8779415\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_16_s4_R1.fastq.gz:23615664\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_16_s5_R1.fastq.gz:23906002\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_16_s6_R1.fastq.gz:23662346\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_17_s1_R1.fastq.gz:5452289\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_17_s2_R1.fastq.gz:5224892\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_17_s3_R1.fastq.gz:4792356\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_17_s4_R1.fastq.gz:13537941\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_17_s5_R1.fastq.gz:13388541\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_17_s6_R1.fastq.gz:13424568\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_18_s1_R1.fastq.gz:7205960\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_18_s2_R1.fastq.gz:7067776\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_18_s3_R1.fastq.gz:6298989\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_18_s4_R1.fastq.gz:17702415\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_18_s5_R1.fastq.gz:17512007\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_18_s6_R1.fastq.gz:17562879\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_1_s1_R1.fastq.gz:6074160\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_1_s2_R1.fastq.gz:6223799\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_1_s3_R1.fastq.gz:5590584\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_1_s4_R1.fastq.gz:15076683\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_1_s5_R1.fastq.gz:14932739\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_1_s6_R1.fastq.gz:14970282\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_2_s1_R1.fastq.gz:6171777\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_2_s2_R1.fastq.gz:5890562\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_2_s3_R1.fastq.gz:5521250\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_2_s4_R1.fastq.gz:14801515\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_2_s5_R1.fastq.gz:14649711\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_2_s6_R1.fastq.gz:14703208\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_3_s1_R1.fastq.gz:6367353\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_3_s2_R1.fastq.gz:6031667\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_3_s3_R1.fastq.gz:5584683\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_3_s4_R1.fastq.gz:15508074\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_3_s5_R1.fastq.gz:15378237\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_3_s6_R1.fastq.gz:15415276\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_4_s1_R1.fastq.gz:5835927\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_4_s2_R1.fastq.gz:5619396\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_4_s3_R1.fastq.gz:5293583\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_4_s4_R1.fastq.gz:14449163\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_4_s5_R1.fastq.gz:14271465\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_4_s6_R1.fastq.gz:14289906\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_5_s1_R1.fastq.gz:6283056\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_5_s2_R1.fastq.gz:6158545\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_5_s3_R1.fastq.gz:5691560\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_5_s4_R1.fastq.gz:15307920\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_5_s5_R1.fastq.gz:15121260\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_5_s6_R1.fastq.gz:15154485\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_6_s1_R1.fastq.gz:5824794\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_6_s2_R1.fastq.gz:5725069\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_6_s3_R1.fastq.gz:5060678\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_6_s4_R1.fastq.gz:14962344\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_6_s5_R1.fastq.gz:14807873\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_6_s6_R1.fastq.gz:14830851\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_7_s1_R1.fastq.gz:6636263\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_7_s2_R1.fastq.gz:6381643\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_7_s3_R1.fastq.gz:5863521\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_7_s4_R1.fastq.gz:16043877\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_7_s5_R1.fastq.gz:15838006\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_7_s6_R1.fastq.gz:15863522\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_8_s1_R1.fastq.gz:5354370\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_8_s2_R1.fastq.gz:5151441\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_8_s3_R1.fastq.gz:4780667\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_8_s4_R1.fastq.gz:12424238\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_8_s5_R1.fastq.gz:12290612\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_8_s6_R1.fastq.gz:12327811\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_9_s1_R1.fastq.gz:9885315\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_9_s2_R1.fastq.gz:9926805\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_9_s3_R1.fastq.gz:8976011\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_9_s4_R1.fastq.gz:24894836\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_9_s5_R1.fastq.gz:24644249\n",
      "Data/MBD/Clean_Data/cleaned_zr1394_9_s6_R1.fastq.gz:24705402\n"
     ]
    }
   ],
   "source": [
    "!zgrep -c \"@HW\" Data/MBD/Clean_Data/*.fastq.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Concatenate read files into one per sample"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: Data/MBD/Clean_Joined_Data: File exists\r\n"
     ]
    }
   ],
   "source": [
    "!mkdir Data/MBD/Clean_Joined_Data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_1_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_1_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_2_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_2_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_3_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_3_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_4_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_4_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_5_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_5_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_6_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_6_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_7_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_7_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_8_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_8_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_9_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_9_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_10_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_10_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_11_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_11_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_12_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_12_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_13_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_13_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_14_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_14_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_15_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_15_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_16_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_16_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_17_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_17_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!cat Data/MBD/Clean_Data/cleaned_*_18_*.fastq.gz > Data/MBD/Clean_Joined_Data/zr1394_18_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Check to see the total number of sequences is the same in the joined file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Data/MBD/Clean_Joined_Data/zr1394_10_cleaned_all.fastq.gz:104694897\n",
      "Data/MBD/Clean_Joined_Data/zr1394_11_cleaned_all.fastq.gz:71728916\n",
      "Data/MBD/Clean_Joined_Data/zr1394_12_cleaned_all.fastq.gz:80542929\n",
      "Data/MBD/Clean_Joined_Data/zr1394_13_cleaned_all.fastq.gz:86407336\n",
      "Data/MBD/Clean_Joined_Data/zr1394_14_cleaned_all.fastq.gz:117183295\n",
      "Data/MBD/Clean_Joined_Data/zr1394_15_cleaned_all.fastq.gz:91725899\n",
      "Data/MBD/Clean_Joined_Data/zr1394_16_cleaned_all.fastq.gz:99156555\n",
      "Data/MBD/Clean_Joined_Data/zr1394_17_cleaned_all.fastq.gz:55820587\n",
      "Data/MBD/Clean_Joined_Data/zr1394_18_cleaned_all.fastq.gz:73350026\n",
      "Data/MBD/Clean_Joined_Data/zr1394_1_cleaned_all.fastq.gz:62868247\n",
      "Data/MBD/Clean_Joined_Data/zr1394_2_cleaned_all.fastq.gz:61738023\n",
      "Data/MBD/Clean_Joined_Data/zr1394_3_cleaned_all.fastq.gz:64285290\n",
      "Data/MBD/Clean_Joined_Data/zr1394_4_cleaned_all.fastq.gz:59759440\n",
      "Data/MBD/Clean_Joined_Data/zr1394_5_cleaned_all.fastq.gz:63716826\n",
      "Data/MBD/Clean_Joined_Data/zr1394_6_cleaned_all.fastq.gz:61211609\n",
      "Data/MBD/Clean_Joined_Data/zr1394_7_cleaned_all.fastq.gz:66626832\n",
      "Data/MBD/Clean_Joined_Data/zr1394_8_cleaned_all.fastq.gz:52329139\n",
      "Data/MBD/Clean_Joined_Data/zr1394_9_cleaned_all.fastq.gz:103032618\n"
     ]
    }
   ],
   "source": [
    "!zgrep -c \"@HW\" Data/MBD/Clean_Joined_Data/*.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!gunzip Data/MBD/Clean_Joined_Data/*.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python [default]",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
