{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Set Running Environment and Programs using Docker\n",
    "\n",
    "## SOFTWARE VERSION \n",
    "\n",
    " * bedtools 2.26.0\n",
    " * bismark 0.15.0  \n",
    " * blast 2.4.0+\n",
    " * bowtie2 2.2.8 \n",
    " * bsmap 2.90\n",
    " * cufflinks 2.1.1\n",
    " * dammit\n",
    " * fastqc 0.11.5\n",
    " * fastx_toolkit 0.0.13  \n",
    " * R 3.2.5  \n",
    " * RStudio Server0.99 \n",
    " * pyrad 3.0.66 \n",
    " * samtools 0.1.19 \n",
    " * stacks 1.40 \n",
    " * tophat 2.1.1 \n",
    " * trimmomatic 0.36  "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Identify data\n",
    "## Path to genome file\n",
    "\n",
    "* /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/20160512/Ostrea_lurida.scafSeq\n",
    "\n",
    "* /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-5k.fa\n",
    "* /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa\n",
    "* /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-30k.fa\n",
    "* /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-40k.fa\n",
    "\n",
    "## Path to MBD files\n",
    "* /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Run QC on the Raw MBD data\n",
    "* Make a new directory for the QC of the raw data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: cannot create directory '/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Raw_MBD_QC': File exists\r\n"
     ]
    }
   ],
   "source": [
    "!mkdir /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Raw_MBD_QC"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "* Run [FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) Version \n",
    "Andrews S. (2010). FastQC: a quality control tool for high throughput sequence data. Available online at: http://www.bioinformatics.babraham.ac.uk/projects/fastqc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of zr1394_10_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_10_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_10_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_10_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_10_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_10_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_10_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_10_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_10_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_10_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_10_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_10_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_10_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Approx 100% complete for zr1394_10_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_10_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_10_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_10_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_10_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_11_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_11_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_11_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_11_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_11_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_11_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_11_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_11_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_11_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_11_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_11_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_11_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_11_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_11_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_11_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_11_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_11_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_11_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_12_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Approx 100% complete for zr1394_12_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_12_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_12_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_12_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_12_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_12_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_12_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_12_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_12_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_12_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_12_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_12_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_12_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_12_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_12_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_12_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_12_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_13_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Approx 100% complete for zr1394_13_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_13_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_13_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_13_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_13_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_13_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_13_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_13_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_13_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_13_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_13_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_13_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_13_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_13_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_13_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_13_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_13_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_14_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_14_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_14_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_14_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_14_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_14_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_14_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_14_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_14_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_14_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_14_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_14_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_14_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_14_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_14_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_14_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_14_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_14_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_15_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_15_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_15_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_15_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_15_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_15_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_15_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_15_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_15_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_15_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_15_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_15_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_15_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_15_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_15_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_15_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_15_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_15_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_16_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_16_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_16_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_16_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_16_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_16_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_16_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_16_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_16_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_16_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_16_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_16_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_16_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_16_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_16_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_16_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_16_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_16_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_17_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_17_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_17_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_17_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_17_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_17_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_17_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_17_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_17_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_17_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_17_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_17_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_17_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_17_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_17_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_17_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_17_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_17_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_18_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_18_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_18_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_18_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_18_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_18_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_18_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_18_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_18_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_18_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_18_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_18_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_18_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_18_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_18_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_18_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_18_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_18_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_1_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_1_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_1_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_1_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_1_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_1_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_1_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_1_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_1_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_1_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_1_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_1_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_1_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_1_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_1_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_1_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_1_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_1_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_2_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Approx 100% complete for zr1394_2_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_2_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_2_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_2_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_2_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_2_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_2_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_2_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_2_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_2_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_2_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_2_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_2_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_2_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_2_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_2_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_2_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_3_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_3_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_3_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_3_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_3_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_3_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_3_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_3_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_3_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_3_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_3_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_3_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_3_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_3_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_3_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_3_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_3_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_3_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_4_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_4_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_4_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_4_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_4_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_4_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_4_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_4_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_4_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_4_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_4_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_4_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_4_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_4_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_4_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_4_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_4_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_4_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_5_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_5_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_5_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_5_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_5_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_5_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_5_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_5_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_5_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_5_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_5_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_5_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_5_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_5_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_5_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_5_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_5_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_5_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_6_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Approx 100% complete for zr1394_6_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_6_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_6_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_6_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_6_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_6_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_6_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_6_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_6_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_6_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_6_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_6_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_6_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_6_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_6_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_6_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_6_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_7_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_7_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_7_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_7_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_7_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_7_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_7_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_7_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_7_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_7_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_7_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_7_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_7_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_7_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_7_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_7_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_7_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_7_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_8_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_8_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_8_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_8_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_8_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_8_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_8_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_8_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_8_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_8_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_8_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_8_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_8_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_8_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_8_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_8_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_8_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_8_s6_R1.fastq.gz\n",
      "Started analysis of zr1394_9_s1_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_9_s1_R1.fastq.gz\n",
      "Analysis complete for zr1394_9_s1_R1.fastq.gz\n",
      "Started analysis of zr1394_9_s2_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_9_s2_R1.fastq.gz\n",
      "Analysis complete for zr1394_9_s2_R1.fastq.gz\n",
      "Started analysis of zr1394_9_s3_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_9_s3_R1.fastq.gz\n",
      "Analysis complete for zr1394_9_s3_R1.fastq.gz\n",
      "Started analysis of zr1394_9_s4_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_9_s4_R1.fastq.gz\n",
      "Analysis complete for zr1394_9_s4_R1.fastq.gz\n",
      "Started analysis of zr1394_9_s5_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_9_s5_R1.fastq.gz\n",
      "Analysis complete for zr1394_9_s5_R1.fastq.gz\n",
      "Started analysis of zr1394_9_s6_R1.fastq.gz\n",
      "Approx 5% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 10% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 15% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 20% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 25% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 30% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 35% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 40% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 45% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 50% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 55% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 60% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 65% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 70% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 75% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 80% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 85% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 90% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Approx 95% complete for zr1394_9_s6_R1.fastq.gz\n",
      "Analysis complete for zr1394_9_s6_R1.fastq.gz\n"
     ]
    }
   ],
   "source": [
    "!fastqc /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/*R1.fastq.gz \\\n",
    " -o /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Raw_MBD_QC"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Trim adapters and low quality sequences to generate cleaned data files\n",
    "* Make a new directory for the cleaned data files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "amkdir /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Loop files to complete trimmomatic on all sample read files\n",
    "* Remove Illumina adapters: ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10\n",
    "* Remove leading low quality or N bases below quality =3: LEADING:3\n",
    "* Remove trailing low quality or N bases below quality =3: TRAILING:3\n",
    "* Scan with a sliding window of 4 bases, cutting when  average quality per base is below 15: SLIDINGWINDOW:4:15\n",
    "* Remove reads below 20bp long: MINLEN:20;\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_10_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_10_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 12129726 Surviving: 10441477 (86.08%) Dropped: 1688249 (13.92%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_10_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_10_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 11482874 Surviving: 10123890 (88.17%) Dropped: 1358984 (11.83%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_10_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_10_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 10547199 Surviving: 9474454 (89.83%) Dropped: 1072745 (10.17%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_10_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_10_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 26306980 Surviving: 24760860 (94.12%) Dropped: 1546120 (5.88%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_10_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_10_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 26573036 Surviving: 25046673 (94.26%) Dropped: 1526363 (5.74%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_10_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_10_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 26384567 Surviving: 24847543 (94.17%) Dropped: 1537024 (5.83%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_11_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_11_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7893266 Surviving: 6852625 (86.82%) Dropped: 1040641 (13.18%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_11_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_11_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7495554 Surviving: 6662076 (88.88%) Dropped: 833478 (11.12%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_11_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_11_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6775729 Surviving: 6125516 (90.40%) Dropped: 650213 (9.60%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_11_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_11_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 18477911 Surviving: 17475776 (94.58%) Dropped: 1002135 (5.42%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_11_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_11_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 18305906 Surviving: 17291115 (94.46%) Dropped: 1014791 (5.54%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_11_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_11_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 18325711 Surviving: 17321808 (94.52%) Dropped: 1003903 (5.48%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_12_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_12_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 9669005 Surviving: 8343594 (86.29%) Dropped: 1325411 (13.71%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_12_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_12_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 9147875 Surviving: 8067647 (88.19%) Dropped: 1080228 (11.81%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_12_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_12_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 8338418 Surviving: 7489976 (89.82%) Dropped: 848442 (10.18%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_12_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_12_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 20188420 Surviving: 19028902 (94.26%) Dropped: 1159518 (5.74%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_12_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_12_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 20006134 Surviving: 18835156 (94.15%) Dropped: 1170978 (5.85%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_12_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_12_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 19958692 Surviving: 18777654 (94.08%) Dropped: 1181038 (5.92%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_13_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_13_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 9840016 Surviving: 8607193 (87.47%) Dropped: 1232823 (12.53%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_13_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_13_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 9358447 Surviving: 8364117 (89.38%) Dropped: 994330 (10.62%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_13_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_13_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 8578615 Surviving: 7803622 (90.97%) Dropped: 774993 (9.03%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_13_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_13_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 21598254 Surviving: 20439482 (94.63%) Dropped: 1158772 (5.37%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_13_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_13_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 21823361 Surviving: 20682194 (94.77%) Dropped: 1141167 (5.23%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_13_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_13_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 21660067 Surviving: 20510728 (94.69%) Dropped: 1149339 (5.31%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_14_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_14_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 13367447 Surviving: 11516118 (86.15%) Dropped: 1851329 (13.85%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_14_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_14_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 12782967 Surviving: 11262884 (88.11%) Dropped: 1520083 (11.89%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_14_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_14_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 11914441 Surviving: 10707861 (89.87%) Dropped: 1206580 (10.13%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_14_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_14_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 29782244 Surviving: 28102235 (94.36%) Dropped: 1680009 (5.64%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_14_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_14_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 29515814 Surviving: 27825679 (94.27%) Dropped: 1690135 (5.73%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_14_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_14_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 29473848 Surviving: 27768518 (94.21%) Dropped: 1705330 (5.79%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_15_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_15_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 10879464 Surviving: 9382482 (86.24%) Dropped: 1496982 (13.76%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_15_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_15_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 10267923 Surviving: 9052599 (88.16%) Dropped: 1215324 (11.84%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_15_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_15_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 9425665 Surviving: 8455241 (89.70%) Dropped: 970424 (10.30%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_15_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_15_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 23084106 Surviving: 21752869 (94.23%) Dropped: 1331237 (5.77%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_15_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_15_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 22853637 Surviving: 21502281 (94.09%) Dropped: 1351356 (5.91%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_15_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_15_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 22920249 Surviving: 21580427 (94.15%) Dropped: 1339822 (5.85%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_16_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_16_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 11242167 Surviving: 9754481 (86.77%) Dropped: 1487686 (13.23%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_16_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_16_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 10639584 Surviving: 9438647 (88.71%) Dropped: 1200937 (11.29%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_16_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_16_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 9717312 Surviving: 8779415 (90.35%) Dropped: 937897 (9.65%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_16_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_16_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 25050434 Surviving: 23615664 (94.27%) Dropped: 1434770 (5.73%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_16_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_16_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 25317275 Surviving: 23906002 (94.43%) Dropped: 1411273 (5.57%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_16_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_16_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 25082606 Surviving: 23662346 (94.34%) Dropped: 1420260 (5.66%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_17_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_17_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6292243 Surviving: 5452289 (86.65%) Dropped: 839954 (13.35%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_17_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_17_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 5878505 Surviving: 5224892 (88.88%) Dropped: 653613 (11.12%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_17_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_17_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 5300767 Surviving: 4792356 (90.41%) Dropped: 508411 (9.59%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_17_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_17_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 14329557 Surviving: 13537941 (94.48%) Dropped: 791616 (5.52%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_17_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_17_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 14186344 Surviving: 13388541 (94.38%) Dropped: 797803 (5.62%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_17_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_17_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 14214584 Surviving: 13424568 (94.44%) Dropped: 790016 (5.56%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_18_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_18_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 8197198 Surviving: 7205960 (87.91%) Dropped: 991238 (12.09%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_18_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_18_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7871675 Surviving: 7067776 (89.79%) Dropped: 803899 (10.21%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_18_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_18_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6898091 Surviving: 6298989 (91.31%) Dropped: 599102 (8.69%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_18_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_18_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 18627445 Surviving: 17702415 (95.03%) Dropped: 925030 (4.97%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_18_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_18_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 18447543 Surviving: 17512007 (94.93%) Dropped: 935536 (5.07%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_18_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_18_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 18484693 Surviving: 17562879 (95.01%) Dropped: 921814 (4.99%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_1_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_1_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6768171 Surviving: 6074160 (89.75%) Dropped: 694011 (10.25%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_1_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_1_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7088611 Surviving: 6223799 (87.80%) Dropped: 864812 (12.20%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_1_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_1_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6127974 Surviving: 5590584 (91.23%) Dropped: 537390 (8.77%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_1_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_1_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15875631 Surviving: 15076683 (94.97%) Dropped: 798948 (5.03%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_1_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_1_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15747941 Surviving: 14932739 (94.82%) Dropped: 815202 (5.18%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_1_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_1_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15775841 Surviving: 14970282 (94.89%) Dropped: 805559 (5.11%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_2_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_2_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7073025 Surviving: 6171777 (87.26%) Dropped: 901248 (12.74%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_2_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_2_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6596118 Surviving: 5890562 (89.30%) Dropped: 705556 (10.70%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_2_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_2_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6079643 Surviving: 5521250 (90.82%) Dropped: 558393 (9.18%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_2_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_2_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15635185 Surviving: 14801515 (94.67%) Dropped: 833670 (5.33%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_2_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_2_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15492802 Surviving: 14649711 (94.56%) Dropped: 843091 (5.44%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_2_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_2_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15538451 Surviving: 14703208 (94.62%) Dropped: 835243 (5.38%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_3_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_3_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7293076 Surviving: 6367353 (87.31%) Dropped: 925723 (12.69%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_3_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_3_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6737811 Surviving: 6031667 (89.52%) Dropped: 706144 (10.48%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_3_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_3_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6144269 Surviving: 5584683 (90.89%) Dropped: 559586 (9.11%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_3_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_3_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 16379997 Surviving: 15508074 (94.68%) Dropped: 871923 (5.32%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_3_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_3_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 16260168 Surviving: 15378237 (94.58%) Dropped: 881931 (5.42%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_3_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_3_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 16286455 Surviving: 15415276 (94.65%) Dropped: 871179 (5.35%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_4_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_4_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6649632 Surviving: 5835927 (87.76%) Dropped: 813705 (12.24%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_4_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_4_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6270620 Surviving: 5619396 (89.61%) Dropped: 651224 (10.39%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_4_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_4_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 5813100 Surviving: 5293583 (91.06%) Dropped: 519517 (8.94%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_4_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_4_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15241034 Surviving: 14449163 (94.80%) Dropped: 791871 (5.20%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_4_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_4_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15080117 Surviving: 14271465 (94.64%) Dropped: 808652 (5.36%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_4_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_4_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15090774 Surviving: 14289906 (94.69%) Dropped: 800868 (5.31%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_5_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_5_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7149560 Surviving: 6283056 (87.88%) Dropped: 866504 (12.12%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_5_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_5_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6855766 Surviving: 6158545 (89.83%) Dropped: 697221 (10.17%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_5_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_5_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6226422 Surviving: 5691560 (91.41%) Dropped: 534862 (8.59%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_5_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_5_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 16122713 Surviving: 15307920 (94.95%) Dropped: 814793 (5.05%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_5_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_5_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15952921 Surviving: 15121260 (94.79%) Dropped: 831661 (5.21%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_5_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_5_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15974422 Surviving: 15154485 (94.87%) Dropped: 819937 (5.13%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_6_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_6_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6614004 Surviving: 5824794 (88.07%) Dropped: 789210 (11.93%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_6_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_6_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6355162 Surviving: 5725069 (90.09%) Dropped: 630093 (9.91%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_6_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_6_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 5529830 Surviving: 5060678 (91.52%) Dropped: 469152 (8.48%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_6_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_6_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15735916 Surviving: 14962344 (95.08%) Dropped: 773572 (4.92%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_6_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_6_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15596312 Surviving: 14807873 (94.94%) Dropped: 788439 (5.06%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_6_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_6_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 15612183 Surviving: 14830851 (95.00%) Dropped: 781332 (5.00%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_7_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_7_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7571742 Surviving: 6636263 (87.65%) Dropped: 935479 (12.35%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_7_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_7_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 7125072 Surviving: 6381643 (89.57%) Dropped: 743429 (10.43%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_7_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_7_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6436872 Surviving: 5863521 (91.09%) Dropped: 573351 (8.91%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_7_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_7_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 16920844 Surviving: 16043877 (94.82%) Dropped: 876967 (5.18%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_7_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_7_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 16731051 Surviving: 15838006 (94.66%) Dropped: 893045 (5.34%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_7_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_7_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 16747458 Surviving: 15863522 (94.72%) Dropped: 883936 (5.28%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_8_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_8_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 6146166 Surviving: 5354370 (87.12%) Dropped: 791796 (12.88%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_8_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_8_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 5775399 Surviving: 5151441 (89.20%) Dropped: 623958 (10.80%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_8_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_8_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 5262335 Surviving: 4780667 (90.85%) Dropped: 481668 (9.15%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_8_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_8_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 13117656 Surviving: 12424238 (94.71%) Dropped: 693418 (5.29%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_8_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_8_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 12994067 Surviving: 12290612 (94.59%) Dropped: 703455 (5.41%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_8_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_8_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 13024791 Surviving: 12327811 (94.65%) Dropped: 696980 (5.35%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_9_s1_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_9_s1_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 11259296 Surviving: 9885315 (87.80%) Dropped: 1373981 (12.20%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_9_s2_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_9_s2_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 11063708 Surviving: 9926805 (89.72%) Dropped: 1136903 (10.28%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_9_s3_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_9_s3_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 9830662 Surviving: 8976011 (91.31%) Dropped: 854651 (8.69%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_9_s4_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_9_s4_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 26183621 Surviving: 24894836 (95.08%) Dropped: 1288785 (4.92%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_9_s5_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_9_s5_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 25958649 Surviving: 24644249 (94.94%) Dropped: 1314400 (5.06%)\n",
      "TrimmomaticSE: Completed successfully\n",
      "TrimmomaticSE: Started with arguments:\n",
      " -threads 16 -phred33 /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/zr1394_9_s6_R1.fastq.gz /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_zr1394_9_s6_R1.fastq.gz ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:20\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTA'\n",
      "Using Long Clipping Sequence: 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC'\n",
      "ILLUMINACLIP: Using 0 prefix pairs, 2 forward/reverse sequences, 0 forward only sequences, 0 reverse only sequences\n",
      "Input Reads: 26004238 Surviving: 24705402 (95.01%) Dropped: 1298836 (4.99%)\n",
      "TrimmomaticSE: Completed successfully\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for file in /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq/*R1.fastq.gz\n",
    "do\n",
    "newname=${file##*/} \n",
    "java -jar /usr/local/bioinformatics/Trimmomatic-0.36/trimmomatic-0.36.jar \\\n",
    "SE \\\n",
    "-threads 16 \\\n",
    "-phred33 \"$file\" \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_$newname \\\n",
    "ILLUMINACLIP:/Users/hollie/Documents/HP_BioInf/Oly_MBD/Notebooks/TruSeq3-SE.fa:2:30:10 \\\n",
    "LEADING:3 \\\n",
    "TRAILING:3 \\\n",
    "SLIDINGWINDOW:4:15 \\\n",
    "MINLEN:20;\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Run QC on the cleaned MBD files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_10_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_10_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_10_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_10_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_10_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_10_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_11_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_11_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_11_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_11_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_11_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_11_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_12_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_12_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_12_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_12_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_12_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_12_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_13_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_13_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_13_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_13_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_13_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_13_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_14_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_14_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_14_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_14_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_14_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_14_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_15_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_15_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_15_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_15_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_15_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_15_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_16_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_16_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_16_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_16_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Approx 100% complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_16_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_16_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_17_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_17_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_17_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_17_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_17_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_17_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_18_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_18_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_18_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_18_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Approx 100% complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_18_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_18_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_1_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_1_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_1_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_1_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_1_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_1_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_2_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_2_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_2_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_2_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_2_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_2_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_3_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_3_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_3_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_3_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_3_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_3_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_4_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_4_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_4_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_4_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_4_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_4_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_5_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_5_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_5_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_5_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_5_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_5_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_6_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_6_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_6_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_6_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_6_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_6_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_7_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_7_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_7_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_7_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Approx 100% complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_7_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_7_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_8_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_8_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_8_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_8_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_8_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_8_s6_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_9_s1_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_9_s2_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_9_s3_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_9_s4_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_9_s5_R1.fastq.gz\n",
      "Started analysis of cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 5% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 10% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 15% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 20% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 25% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 30% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 35% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 40% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 45% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 50% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 55% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 60% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 65% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 70% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 75% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 80% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 85% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 90% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Approx 95% complete for cleaned_zr1394_9_s6_R1.fastq.gz\n",
      "Analysis complete for cleaned_zr1394_9_s6_R1.fastq.gz\n"
     ]
    }
   ],
   "source": [
    "!fastqc /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/*R1.fastq.gz \\\n",
    " -o /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/cleaned_MBD_QC"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Concatenate read file into one per sample"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "loop through files by sample and concatenate 6 files per sample into a single file per sample"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!mkdir /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_1_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_1_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_2_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_2_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_3_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_3_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_4_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_4_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_5_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_5_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_6_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_6_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_7_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_7_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_8_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_8_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_9_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_9_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_10_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_10_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_11_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_11_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_12_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_12_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_13_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_13_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_14_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_14_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_15_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_15_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_16_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_16_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_17_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_17_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned/cleaned_*_18_*.fastq.gz \\\n",
    "> /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_18_cleaned_all.fastq.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Unzip all cleaned concatenated read files for use in methylation call analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "^C\r\n"
     ]
    }
   ],
   "source": [
    "!gunzip /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/*fastq.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "# Run BSMAP "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/BSMap_Output\n"
     ]
    }
   ],
   "source": [
    "cd /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/BSMap_Output/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Process is interrupted.\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for file in /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_7_cleaned_all.fastq\n",
    "do \n",
    "newname=${file##*/}\n",
    "newname2=${newname/_cleaned_all.fastq/}\n",
    "bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/$newname \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o 10K_bsmap_out_$newname2.sam \\\n",
    "-p 8;\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bsmap] @Tue Oct 25 07:10:06 2016 \tloading reference file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \t(format: FASTA)\n",
      "[bsmap] @Tue Oct 25 07:10:10 2016 \t8733 reference seqs loaded, total size 131223850 bp. 4 secs passed\n",
      "[bsmap] @Tue Oct 25 07:10:17 2016 \tcreate seed table. 11 secs passed\n",
      "[bsmap] @Tue Oct 25 07:10:17 2016 \tSingle-end alignment(8 threads),\n",
      "\tInput read file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_1_cleaned_all.fastq \t(format: FASTQ)\n",
      "\tOutput file: bsmap_out_zr1394_1_10K.sam\t (format: SAM)\n",
      "[bsmap] @Tue Oct 25 07:35:37 2016 \ttotal reads: 62868247 \ttotal time:  1531 secs\n",
      "\taligned reads: 6978336 (11.1%), unique reads: 4145996 (6.6%), non-unique reads: 2832340 (4.5%)\n"
     ]
    }
   ],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_1_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_1_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bsmap] @Tue Oct 25 07:35:38 2016 \tloading reference file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \t(format: FASTA)\n",
      "[bsmap] @Tue Oct 25 07:35:43 2016 \t8733 reference seqs loaded, total size 131223850 bp. 5 secs passed\n",
      "[bsmap] @Tue Oct 25 07:35:49 2016 \tcreate seed table. 11 secs passed\n",
      "[bsmap] @Tue Oct 25 07:35:49 2016 \tSingle-end alignment(8 threads),\n",
      "\tInput read file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_2_cleaned_all.fastq \t(format: FASTQ)\n",
      "\tOutput file: bsmap_out_zr1394_2_10K.sam\t (format: SAM)\n",
      "[bsmap] @Tue Oct 25 08:00:38 2016 \ttotal reads: 61738023 \ttotal time:  1500 secs\n",
      "\taligned reads: 6588662 (10.7%), unique reads: 3947767 (6.4%), non-unique reads: 2640895 (4.3%)\n"
     ]
    }
   ],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_2_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_2_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bsmap] @Tue Oct 25 08:00:39 2016 \tloading reference file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \t(format: FASTA)\n",
      "[bsmap] @Tue Oct 25 08:00:42 2016 \t8733 reference seqs loaded, total size 131223850 bp. 3 secs passed\n",
      "[bsmap] @Tue Oct 25 08:00:48 2016 \tcreate seed table. 9 secs passed\n",
      "[bsmap] @Tue Oct 25 08:00:48 2016 \tSingle-end alignment(8 threads),\n",
      "\tInput read file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_3_cleaned_all.fastq \t(format: FASTQ)\n",
      "\tOutput file: bsmap_out_zr1394_3_10K.sam\t (format: SAM)\n",
      "[bsmap] @Tue Oct 25 08:26:35 2016 \ttotal reads: 64285290 \ttotal time:  1556 secs\n",
      "\taligned reads: 6934635 (10.8%), unique reads: 4107736 (6.4%), non-unique reads: 2826899 (4.4%)\n"
     ]
    }
   ],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_3_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_3_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bsmap] @Tue Oct 25 08:26:36 2016 \tloading reference file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \t(format: FASTA)\n",
      "[bsmap] @Tue Oct 25 08:26:40 2016 \t8733 reference seqs loaded, total size 131223850 bp. 4 secs passed\n",
      "[bsmap] @Tue Oct 25 08:26:47 2016 \tcreate seed table. 11 secs passed\n",
      "[bsmap] @Tue Oct 25 08:26:47 2016 \tSingle-end alignment(8 threads),\n",
      "\tInput read file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_4_cleaned_all.fastq \t(format: FASTQ)\n",
      "\tOutput file: bsmap_out_zr1394_4_10K.sam\t (format: SAM)\n",
      "[bsmap] @Tue Oct 25 08:50:43 2016 \ttotal reads: 59759440 \ttotal time:  1447 secs\n",
      "\taligned reads: 6508956 (10.9%), unique reads: 3864202 (6.5%), non-unique reads: 2644754 (4.4%)\n"
     ]
    }
   ],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_4_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_4_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bsmap] @Tue Oct 25 08:50:43 2016 \tloading reference file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \t(format: FASTA)\n",
      "[bsmap] @Tue Oct 25 08:50:47 2016 \t8733 reference seqs loaded, total size 131223850 bp. 4 secs passed\n",
      "[bsmap] @Tue Oct 25 08:50:53 2016 \tcreate seed table. 10 secs passed\n",
      "[bsmap] @Tue Oct 25 08:50:53 2016 \tSingle-end alignment(8 threads),\n",
      "\tInput read file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_5_cleaned_all.fastq \t(format: FASTQ)\n",
      "\tOutput file: bsmap_out_zr1394_5_10K.sam\t (format: SAM)\n",
      "[bsmap] @Tue Oct 25 09:16:28 2016 \ttotal reads: 63716826 \ttotal time:  1545 secs\n",
      "\taligned reads: 6973839 (10.9%), unique reads: 4195247 (6.6%), non-unique reads: 2778592 (4.4%)\n"
     ]
    }
   ],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_5_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_5_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bsmap] @Tue Oct 25 09:16:28 2016 \tloading reference file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \t(format: FASTA)\n",
      "[bsmap] @Tue Oct 25 09:16:33 2016 \t8733 reference seqs loaded, total size 131223850 bp. 5 secs passed\n",
      "[bsmap] @Tue Oct 25 09:16:40 2016 \tcreate seed table. 12 secs passed\n",
      "[bsmap] @Tue Oct 25 09:16:40 2016 \tSingle-end alignment(8 threads),\n",
      "\tInput read file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_6_cleaned_all.fastq \t(format: FASTQ)\n",
      "\tOutput file: bsmap_out_zr1394_6_10K.sam\t (format: SAM)\n",
      "^C\n"
     ]
    }
   ],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_6_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_6_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bsmap] @Tue Oct 25 17:03:42 2016 \tloading reference file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \t(format: FASTA)\n",
      "[bsmap] @Tue Oct 25 17:03:47 2016 \t8733 reference seqs loaded, total size 131223850 bp. 5 secs passed\n",
      "^C\n"
     ]
    }
   ],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_7_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_7_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bsmap] @Tue Oct 25 17:03:49 2016 \tloading reference file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \t(format: FASTA)\n",
      "[bsmap] @Tue Oct 25 17:03:53 2016 \t8733 reference seqs loaded, total size 131223850 bp. 4 secs passed\n",
      "[bsmap] @Tue Oct 25 17:03:59 2016 \tcreate seed table. 10 secs passed\n",
      "[bsmap] @Tue Oct 25 17:03:59 2016 \tSingle-end alignment(8 threads),\n",
      "\tInput read file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_8_cleaned_all.fastq \t(format: FASTQ)\n",
      "\tOutput file: bsmap_out_zr1394_8_10K.sam\t (format: SAM)\n",
      "[bsmap] @Tue Oct 25 17:25:14 2016 \ttotal reads: 52329139 \ttotal time:  1285 secs\n",
      "\taligned reads: 5784667 (11.1%), unique reads: 3429646 (6.6%), non-unique reads: 2355021 (4.5%)\n"
     ]
    }
   ],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_8_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_8_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[bsmap] @Tue Oct 25 17:25:15 2016 \tloading reference file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \t(format: FASTA)\n",
      "[bsmap] @Tue Oct 25 17:25:20 2016 \t8733 reference seqs loaded, total size 131223850 bp. 5 secs passed\n",
      "[bsmap] @Tue Oct 25 17:25:26 2016 \tcreate seed table. 11 secs passed\n",
      "[bsmap] @Tue Oct 25 17:25:26 2016 \tSingle-end alignment(8 threads),\n",
      "\tInput read file: /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_9_cleaned_all.fastq \t(format: FASTQ)\n",
      "\tOutput file: bsmap_out_zr1394_9_10K.sam\t (format: SAM)\n"
     ]
    }
   ],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_9_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_9_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_10_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_10_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_11_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_11_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_12_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_12_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_13_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_13_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_14_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_14_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_15_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_15_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_16_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_16_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_17_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_17_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!bsmap -a /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/20160203_mbdseq_cleaned_joined/zr1394_18_cleaned_all.fastq \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-o bsmap_out_zr1394_18_10K.sam \\\n",
    "-p 8"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Calculate Methylation of CpG Loci"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "for file in /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/MBD/BSMap_Output/*.sam\n",
    "do \n",
    "newname=${file##*/}\n",
    "    python methratio.py \\\n",
    "-d /Users/hollie/Documents/HP_BioInf/Oly_MBD/Data/Genome/Filtered_Genome/Ostrea_lurida-Scaff-10k.fa \\\n",
    "-u -z -g \\\n",
    "-o methratio_out_$newname.txt \\\n",
    "-s samtools \\\n",
    "bsmap_out_$newname.sam;\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "SyntaxError",
     "evalue": "invalid syntax (<ipython-input-30-8a2a2d8c1af0>, line 1)",
     "output_type": "error",
     "traceback": [
      "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-30-8a2a2d8c1af0>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0m\n\u001b[0;31m    python methratio.py\u001b[0m\n\u001b[0m                   ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
     ]
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python [default]",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
